[2020-03-25 04:08:52] local.ERROR: Error: Class 'App\Http\Controllers\User' not found in C:\xampp\htdocs\lumenapijwt\app\Http\Controllers\AuthController.php:20
Stack trace:
#0 [internal function]: App\Http\Controllers\AuthController->register(Object(Laravel\Lumen\Http\Request))
#1 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\container\BoundMethod.php(32): call_user_func_array(Array, Array)
#2 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\container\Util.php(36): Illuminate\Container\BoundMethod::Illuminate\Container\{closure}()
#3 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\container\BoundMethod.php(90): Illuminate\Container\Util::unwrapIfClosure(Object(Closure))
#4 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\container\BoundMethod.php(34): Illuminate\Container\BoundMethod::callBoundMethod(Object(Laravel\Lumen\Application), Array, Object(Closure))
#5 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\container\Container.php(592): Illuminate\Container\BoundMethod::call(Object(Laravel\Lumen\Application), Array, Array, NULL)
#6 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(376): Illuminate\Container\Container->call(Array, Array)
#7 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(342): Laravel\Lumen\Application->callControllerCallable(Array, Array)
#8 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(316): Laravel\Lumen\Application->callLumenController(Object(App\Http\Controllers\AuthController), 'register', Array)
#9 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(278): Laravel\Lumen\Application->callControllerAction(Array)
#10 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(263): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#11 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(165): Laravel\Lumen\Application->handleFoundRoute(Array)
#12 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(416): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}(Object(Laravel\Lumen\Http\Request))
#13 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(171): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#14 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(108): Laravel\Lumen\Application->dispatch(NULL)
#15 C:\xampp\htdocs\lumenapijwt\public\index.php(28): Laravel\Lumen\Application->run()
#16 {main} {"exception":"[object] (Error(code: 0): Class 'App\\Http\\Controllers\\User' not found at C:\\xampp\\htdocs\\lumenapijwt\\app\\Http\\Controllers\\AuthController.php:20)
[stacktrace]
#0 [internal function]: App\\Http\\Controllers\\AuthController->register(Object(Laravel\\Lumen\\Http\\Request))
#1 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\container\\BoundMethod.php(32): call_user_func_array(Array, Array)
#2 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\container\\Util.php(36): Illuminate\\Container\\BoundMethod::Illuminate\\Container\\{closure}()
#3 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\container\\BoundMethod.php(90): Illuminate\\Container\\Util::unwrapIfClosure(Object(Closure))
#4 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\container\\BoundMethod.php(34): Illuminate\\Container\\BoundMethod::callBoundMethod(Object(Laravel\\Lumen\\Application), Array, Object(Closure))
#5 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\container\\Container.php(592): Illuminate\\Container\\BoundMethod::call(Object(Laravel\\Lumen\\Application), Array, Array, NULL)
#6 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(376): Illuminate\\Container\\Container->call(Array, Array)
#7 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(342): Laravel\\Lumen\\Application->callControllerCallable(Array, Array)
#8 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(316): Laravel\\Lumen\\Application->callLumenController(Object(App\\Http\\Controllers\\AuthController), 'register', Array)
#9 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(278): Laravel\\Lumen\\Application->callControllerAction(Array)
#10 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(263): Laravel\\Lumen\\Application->callActionOnArrayBasedRoute(Array)
#11 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(165): Laravel\\Lumen\\Application->handleFoundRoute(Array)
#12 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(416): Laravel\\Lumen\\Application->Laravel\\Lumen\\Concerns\\{closure}(Object(Laravel\\Lumen\\Http\\Request))
#13 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(171): Laravel\\Lumen\\Application->sendThroughPipeline(Array, Object(Closure))
#14 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(108): Laravel\\Lumen\\Application->dispatch(NULL)
#15 C:\\xampp\\htdocs\\lumenapijwt\\public\\index.php(28): Laravel\\Lumen\\Application->run()
#16 {main}
"} 
[2020-03-25 04:12:33] local.ERROR: Symfony\Component\Console\Exception\NamespaceNotFoundException: There are no commands defined in the "jwt" namespace. in C:\xampp\htdocs\lumenapijwt\vendor\symfony\console\Application.php:574
Stack trace:
#0 C:\xampp\htdocs\lumenapijwt\vendor\symfony\console\Application.php(625): Symfony\Component\Console\Application->findNamespace('jwt')
#1 C:\xampp\htdocs\lumenapijwt\vendor\symfony\console\Application.php(228): Symfony\Component\Console\Application->find('jwt:secret')
#2 C:\xampp\htdocs\lumenapijwt\vendor\symfony\console\Application.php(140): Symfony\Component\Console\Application->doRun(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#3 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\console\Application.php(93): Symfony\Component\Console\Application->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#4 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Console\Kernel.php(116): Illuminate\Console\Application->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#5 C:\xampp\htdocs\lumenapijwt\artisan(35): Laravel\Lumen\Console\Kernel->handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#6 {main} {"exception":"[object] (Symfony\\Component\\Console\\Exception\\NamespaceNotFoundException(code: 0): There are no commands defined in the \"jwt\" namespace. at C:\\xampp\\htdocs\\lumenapijwt\\vendor\\symfony\\console\\Application.php:574)
[stacktrace]
#0 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\symfony\\console\\Application.php(625): Symfony\\Component\\Console\\Application->findNamespace('jwt')
#1 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\symfony\\console\\Application.php(228): Symfony\\Component\\Console\\Application->find('jwt:secret')
#2 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\symfony\\console\\Application.php(140): Symfony\\Component\\Console\\Application->doRun(Object(Symfony\\Component\\Console\\Input\\ArgvInput), Object(Symfony\\Component\\Console\\Output\\ConsoleOutput))
#3 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\console\\Application.php(93): Symfony\\Component\\Console\\Application->run(Object(Symfony\\Component\\Console\\Input\\ArgvInput), Object(Symfony\\Component\\Console\\Output\\ConsoleOutput))
#4 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Console\\Kernel.php(116): Illuminate\\Console\\Application->run(Object(Symfony\\Component\\Console\\Input\\ArgvInput), Object(Symfony\\Component\\Console\\Output\\ConsoleOutput))
#5 C:\\xampp\\htdocs\\lumenapijwt\\artisan(35): Laravel\\Lumen\\Console\\Kernel->handle(Object(Symfony\\Component\\Console\\Input\\ArgvInput), Object(Symfony\\Component\\Console\\Output\\ConsoleOutput))
#6 {main}
"} 
[2020-03-25 04:19:06] local.ERROR: Error: Class 'App\Http\Controllers\Auth' not found in C:\xampp\htdocs\lumenapijwt\app\Http\Controllers\AuthController.php:49
Stack trace:
#0 [internal function]: App\Http\Controllers\AuthController->login(Object(Laravel\Lumen\Http\Request))
#1 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\container\BoundMethod.php(32): call_user_func_array(Array, Array)
#2 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\container\Util.php(36): Illuminate\Container\BoundMethod::Illuminate\Container\{closure}()
#3 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\container\BoundMethod.php(90): Illuminate\Container\Util::unwrapIfClosure(Object(Closure))
#4 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\container\BoundMethod.php(34): Illuminate\Container\BoundMethod::callBoundMethod(Object(Laravel\Lumen\Application), Array, Object(Closure))
#5 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\container\Container.php(592): Illuminate\Container\BoundMethod::call(Object(Laravel\Lumen\Application), Array, Array, NULL)
#6 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(376): Illuminate\Container\Container->call(Array, Array)
#7 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(342): Laravel\Lumen\Application->callControllerCallable(Array, Array)
#8 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(316): Laravel\Lumen\Application->callLumenController(Object(App\Http\Controllers\AuthController), 'login', Array)
#9 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(278): Laravel\Lumen\Application->callControllerAction(Array)
#10 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(263): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#11 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(165): Laravel\Lumen\Application->handleFoundRoute(Array)
#12 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(416): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}(Object(Laravel\Lumen\Http\Request))
#13 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(171): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#14 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(108): Laravel\Lumen\Application->dispatch(NULL)
#15 C:\xampp\htdocs\lumenapijwt\public\index.php(28): Laravel\Lumen\Application->run()
#16 {main} {"exception":"[object] (Error(code: 0): Class 'App\\Http\\Controllers\\Auth' not found at C:\\xampp\\htdocs\\lumenapijwt\\app\\Http\\Controllers\\AuthController.php:49)
[stacktrace]
#0 [internal function]: App\\Http\\Controllers\\AuthController->login(Object(Laravel\\Lumen\\Http\\Request))
#1 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\container\\BoundMethod.php(32): call_user_func_array(Array, Array)
#2 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\container\\Util.php(36): Illuminate\\Container\\BoundMethod::Illuminate\\Container\\{closure}()
#3 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\container\\BoundMethod.php(90): Illuminate\\Container\\Util::unwrapIfClosure(Object(Closure))
#4 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\container\\BoundMethod.php(34): Illuminate\\Container\\BoundMethod::callBoundMethod(Object(Laravel\\Lumen\\Application), Array, Object(Closure))
#5 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\container\\Container.php(592): Illuminate\\Container\\BoundMethod::call(Object(Laravel\\Lumen\\Application), Array, Array, NULL)
#6 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(376): Illuminate\\Container\\Container->call(Array, Array)
#7 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(342): Laravel\\Lumen\\Application->callControllerCallable(Array, Array)
#8 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(316): Laravel\\Lumen\\Application->callLumenController(Object(App\\Http\\Controllers\\AuthController), 'login', Array)
#9 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(278): Laravel\\Lumen\\Application->callControllerAction(Array)
#10 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(263): Laravel\\Lumen\\Application->callActionOnArrayBasedRoute(Array)
#11 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(165): Laravel\\Lumen\\Application->handleFoundRoute(Array)
#12 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(416): Laravel\\Lumen\\Application->Laravel\\Lumen\\Concerns\\{closure}(Object(Laravel\\Lumen\\Http\\Request))
#13 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(171): Laravel\\Lumen\\Application->sendThroughPipeline(Array, Object(Closure))
#14 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(108): Laravel\\Lumen\\Application->dispatch(NULL)
#15 C:\\xampp\\htdocs\\lumenapijwt\\public\\index.php(28): Laravel\\Lumen\\Application->run()
#16 {main}
"} 
[2020-03-25 04:20:20] local.ERROR: Error: Class 'App\Http\Controllers\Auth' not found in C:\xampp\htdocs\lumenapijwt\app\Http\Controllers\AuthController.php:49
Stack trace:
#0 [internal function]: App\Http\Controllers\AuthController->login(Object(Laravel\Lumen\Http\Request))
#1 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\container\BoundMethod.php(32): call_user_func_array(Array, Array)
#2 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\container\Util.php(36): Illuminate\Container\BoundMethod::Illuminate\Container\{closure}()
#3 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\container\BoundMethod.php(90): Illuminate\Container\Util::unwrapIfClosure(Object(Closure))
#4 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\container\BoundMethod.php(34): Illuminate\Container\BoundMethod::callBoundMethod(Object(Laravel\Lumen\Application), Array, Object(Closure))
#5 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\container\Container.php(592): Illuminate\Container\BoundMethod::call(Object(Laravel\Lumen\Application), Array, Array, NULL)
#6 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(376): Illuminate\Container\Container->call(Array, Array)
#7 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(342): Laravel\Lumen\Application->callControllerCallable(Array, Array)
#8 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(316): Laravel\Lumen\Application->callLumenController(Object(App\Http\Controllers\AuthController), 'login', Array)
#9 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(278): Laravel\Lumen\Application->callControllerAction(Array)
#10 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(263): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#11 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(165): Laravel\Lumen\Application->handleFoundRoute(Array)
#12 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(416): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}(Object(Laravel\Lumen\Http\Request))
#13 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(171): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#14 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(108): Laravel\Lumen\Application->dispatch(NULL)
#15 C:\xampp\htdocs\lumenapijwt\public\index.php(28): Laravel\Lumen\Application->run()
#16 {main} {"exception":"[object] (Error(code: 0): Class 'App\\Http\\Controllers\\Auth' not found at C:\\xampp\\htdocs\\lumenapijwt\\app\\Http\\Controllers\\AuthController.php:49)
[stacktrace]
#0 [internal function]: App\\Http\\Controllers\\AuthController->login(Object(Laravel\\Lumen\\Http\\Request))
#1 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\container\\BoundMethod.php(32): call_user_func_array(Array, Array)
#2 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\container\\Util.php(36): Illuminate\\Container\\BoundMethod::Illuminate\\Container\\{closure}()
#3 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\container\\BoundMethod.php(90): Illuminate\\Container\\Util::unwrapIfClosure(Object(Closure))
#4 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\container\\BoundMethod.php(34): Illuminate\\Container\\BoundMethod::callBoundMethod(Object(Laravel\\Lumen\\Application), Array, Object(Closure))
#5 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\container\\Container.php(592): Illuminate\\Container\\BoundMethod::call(Object(Laravel\\Lumen\\Application), Array, Array, NULL)
#6 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(376): Illuminate\\Container\\Container->call(Array, Array)
#7 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(342): Laravel\\Lumen\\Application->callControllerCallable(Array, Array)
#8 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(316): Laravel\\Lumen\\Application->callLumenController(Object(App\\Http\\Controllers\\AuthController), 'login', Array)
#9 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(278): Laravel\\Lumen\\Application->callControllerAction(Array)
#10 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(263): Laravel\\Lumen\\Application->callActionOnArrayBasedRoute(Array)
#11 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(165): Laravel\\Lumen\\Application->handleFoundRoute(Array)
#12 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(416): Laravel\\Lumen\\Application->Laravel\\Lumen\\Concerns\\{closure}(Object(Laravel\\Lumen\\Http\\Request))
#13 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(171): Laravel\\Lumen\\Application->sendThroughPipeline(Array, Object(Closure))
#14 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(108): Laravel\\Lumen\\Application->dispatch(NULL)
#15 C:\\xampp\\htdocs\\lumenapijwt\\public\\index.php(28): Laravel\\Lumen\\Application->run()
#16 {main}
"} 
[2020-03-25 04:22:19] local.ERROR: Error: Class 'App\Http\Controllers\Auth' not found in C:\xampp\htdocs\lumenapijwt\app\Http\Controllers\AuthController.php:50
Stack trace:
#0 [internal function]: App\Http\Controllers\AuthController->login(Object(Laravel\Lumen\Http\Request))
#1 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\container\BoundMethod.php(32): call_user_func_array(Array, Array)
#2 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\container\Util.php(36): Illuminate\Container\BoundMethod::Illuminate\Container\{closure}()
#3 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\container\BoundMethod.php(90): Illuminate\Container\Util::unwrapIfClosure(Object(Closure))
#4 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\container\BoundMethod.php(34): Illuminate\Container\BoundMethod::callBoundMethod(Object(Laravel\Lumen\Application), Array, Object(Closure))
#5 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\container\Container.php(592): Illuminate\Container\BoundMethod::call(Object(Laravel\Lumen\Application), Array, Array, NULL)
#6 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(376): Illuminate\Container\Container->call(Array, Array)
#7 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(342): Laravel\Lumen\Application->callControllerCallable(Array, Array)
#8 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(316): Laravel\Lumen\Application->callLumenController(Object(App\Http\Controllers\AuthController), 'login', Array)
#9 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(278): Laravel\Lumen\Application->callControllerAction(Array)
#10 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(263): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#11 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(165): Laravel\Lumen\Application->handleFoundRoute(Array)
#12 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(416): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}(Object(Laravel\Lumen\Http\Request))
#13 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(171): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#14 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(108): Laravel\Lumen\Application->dispatch(NULL)
#15 C:\xampp\htdocs\lumenapijwt\public\index.php(28): Laravel\Lumen\Application->run()
#16 {main} {"exception":"[object] (Error(code: 0): Class 'App\\Http\\Controllers\\Auth' not found at C:\\xampp\\htdocs\\lumenapijwt\\app\\Http\\Controllers\\AuthController.php:50)
[stacktrace]
#0 [internal function]: App\\Http\\Controllers\\AuthController->login(Object(Laravel\\Lumen\\Http\\Request))
#1 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\container\\BoundMethod.php(32): call_user_func_array(Array, Array)
#2 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\container\\Util.php(36): Illuminate\\Container\\BoundMethod::Illuminate\\Container\\{closure}()
#3 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\container\\BoundMethod.php(90): Illuminate\\Container\\Util::unwrapIfClosure(Object(Closure))
#4 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\container\\BoundMethod.php(34): Illuminate\\Container\\BoundMethod::callBoundMethod(Object(Laravel\\Lumen\\Application), Array, Object(Closure))
#5 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\container\\Container.php(592): Illuminate\\Container\\BoundMethod::call(Object(Laravel\\Lumen\\Application), Array, Array, NULL)
#6 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(376): Illuminate\\Container\\Container->call(Array, Array)
#7 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(342): Laravel\\Lumen\\Application->callControllerCallable(Array, Array)
#8 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(316): Laravel\\Lumen\\Application->callLumenController(Object(App\\Http\\Controllers\\AuthController), 'login', Array)
#9 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(278): Laravel\\Lumen\\Application->callControllerAction(Array)
#10 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(263): Laravel\\Lumen\\Application->callActionOnArrayBasedRoute(Array)
#11 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(165): Laravel\\Lumen\\Application->handleFoundRoute(Array)
#12 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(416): Laravel\\Lumen\\Application->Laravel\\Lumen\\Concerns\\{closure}(Object(Laravel\\Lumen\\Http\\Request))
#13 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(171): Laravel\\Lumen\\Application->sendThroughPipeline(Array, Object(Closure))
#14 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(108): Laravel\\Lumen\\Application->dispatch(NULL)
#15 C:\\xampp\\htdocs\\lumenapijwt\\public\\index.php(28): Laravel\\Lumen\\Application->run()
#16 {main}
"} 
[2020-03-25 04:23:22] local.ERROR: BadMethodCallException: Call to undefined method App\User::middleware() in C:\xampp\htdocs\lumenapijwt\vendor\illuminate\support\Traits\ForwardsCalls.php:50
Stack trace:
#0 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\support\Traits\ForwardsCalls.php(36): Illuminate\Database\Eloquent\Model::throwBadMethodCallException('middleware')
#1 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\database\Eloquent\Model.php(1662): Illuminate\Database\Eloquent\Model->forwardCallTo(Object(Illuminate\Database\Eloquent\Builder), 'middleware', Array)
#2 C:\xampp\htdocs\lumenapijwt\app\User.php(66): Illuminate\Database\Eloquent\Model->__call('middleware', Array)
#3 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\auth\EloquentUserProvider.php(183): App\User->__construct()
#4 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\auth\EloquentUserProvider.php(170): Illuminate\Auth\EloquentUserProvider->createModel()
#5 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\auth\EloquentUserProvider.php(117): Illuminate\Auth\EloquentUserProvider->newModelQuery()
#6 C:\xampp\htdocs\lumenapijwt\vendor\tymon\jwt-auth\src\JWTGuard.php(124): Illuminate\Auth\EloquentUserProvider->retrieveByCredentials(Array)
#7 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\auth\AuthManager.php(307): Tymon\JWTAuth\JWTGuard->attempt(Array)
#8 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\support\Facades\Facade.php(261): Illuminate\Auth\AuthManager->__call('attempt', Array)
#9 C:\xampp\htdocs\lumenapijwt\app\Http\Controllers\AuthController.php(50): Illuminate\Support\Facades\Facade::__callStatic('attempt', Array)
#10 [internal function]: App\Http\Controllers\AuthController->login(Object(Laravel\Lumen\Http\Request))
#11 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\container\BoundMethod.php(32): call_user_func_array(Array, Array)
#12 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\container\Util.php(36): Illuminate\Container\BoundMethod::Illuminate\Container\{closure}()
#13 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\container\BoundMethod.php(90): Illuminate\Container\Util::unwrapIfClosure(Object(Closure))
#14 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\container\BoundMethod.php(34): Illuminate\Container\BoundMethod::callBoundMethod(Object(Laravel\Lumen\Application), Array, Object(Closure))
#15 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\container\Container.php(592): Illuminate\Container\BoundMethod::call(Object(Laravel\Lumen\Application), Array, Array, NULL)
#16 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(376): Illuminate\Container\Container->call(Array, Array)
#17 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(342): Laravel\Lumen\Application->callControllerCallable(Array, Array)
#18 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(316): Laravel\Lumen\Application->callLumenController(Object(App\Http\Controllers\AuthController), 'login', Array)
#19 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(278): Laravel\Lumen\Application->callControllerAction(Array)
#20 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(263): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#21 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(165): Laravel\Lumen\Application->handleFoundRoute(Array)
#22 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(416): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}(Object(Laravel\Lumen\Http\Request))
#23 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(171): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#24 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(108): Laravel\Lumen\Application->dispatch(NULL)
#25 C:\xampp\htdocs\lumenapijwt\public\index.php(28): Laravel\Lumen\Application->run()
#26 {main} {"exception":"[object] (BadMethodCallException(code: 0): Call to undefined method App\\User::middleware() at C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\support\\Traits\\ForwardsCalls.php:50)
[stacktrace]
#0 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\support\\Traits\\ForwardsCalls.php(36): Illuminate\\Database\\Eloquent\\Model::throwBadMethodCallException('middleware')
#1 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\database\\Eloquent\\Model.php(1662): Illuminate\\Database\\Eloquent\\Model->forwardCallTo(Object(Illuminate\\Database\\Eloquent\\Builder), 'middleware', Array)
#2 C:\\xampp\\htdocs\\lumenapijwt\\app\\User.php(66): Illuminate\\Database\\Eloquent\\Model->__call('middleware', Array)
#3 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\auth\\EloquentUserProvider.php(183): App\\User->__construct()
#4 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\auth\\EloquentUserProvider.php(170): Illuminate\\Auth\\EloquentUserProvider->createModel()
#5 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\auth\\EloquentUserProvider.php(117): Illuminate\\Auth\\EloquentUserProvider->newModelQuery()
#6 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\tymon\\jwt-auth\\src\\JWTGuard.php(124): Illuminate\\Auth\\EloquentUserProvider->retrieveByCredentials(Array)
#7 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\auth\\AuthManager.php(307): Tymon\\JWTAuth\\JWTGuard->attempt(Array)
#8 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\support\\Facades\\Facade.php(261): Illuminate\\Auth\\AuthManager->__call('attempt', Array)
#9 C:\\xampp\\htdocs\\lumenapijwt\\app\\Http\\Controllers\\AuthController.php(50): Illuminate\\Support\\Facades\\Facade::__callStatic('attempt', Array)
#10 [internal function]: App\\Http\\Controllers\\AuthController->login(Object(Laravel\\Lumen\\Http\\Request))
#11 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\container\\BoundMethod.php(32): call_user_func_array(Array, Array)
#12 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\container\\Util.php(36): Illuminate\\Container\\BoundMethod::Illuminate\\Container\\{closure}()
#13 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\container\\BoundMethod.php(90): Illuminate\\Container\\Util::unwrapIfClosure(Object(Closure))
#14 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\container\\BoundMethod.php(34): Illuminate\\Container\\BoundMethod::callBoundMethod(Object(Laravel\\Lumen\\Application), Array, Object(Closure))
#15 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\container\\Container.php(592): Illuminate\\Container\\BoundMethod::call(Object(Laravel\\Lumen\\Application), Array, Array, NULL)
#16 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(376): Illuminate\\Container\\Container->call(Array, Array)
#17 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(342): Laravel\\Lumen\\Application->callControllerCallable(Array, Array)
#18 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(316): Laravel\\Lumen\\Application->callLumenController(Object(App\\Http\\Controllers\\AuthController), 'login', Array)
#19 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(278): Laravel\\Lumen\\Application->callControllerAction(Array)
#20 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(263): Laravel\\Lumen\\Application->callActionOnArrayBasedRoute(Array)
#21 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(165): Laravel\\Lumen\\Application->handleFoundRoute(Array)
#22 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(416): Laravel\\Lumen\\Application->Laravel\\Lumen\\Concerns\\{closure}(Object(Laravel\\Lumen\\Http\\Request))
#23 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(171): Laravel\\Lumen\\Application->sendThroughPipeline(Array, Object(Closure))
#24 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(108): Laravel\\Lumen\\Application->dispatch(NULL)
#25 C:\\xampp\\htdocs\\lumenapijwt\\public\\index.php(28): Laravel\\Lumen\\Application->run()
#26 {main}
"} 
[2020-03-25 04:24:27] local.ERROR: Error: Class 'App\Http\Controllers\Auth' not found in C:\xampp\htdocs\lumenapijwt\app\Http\Controllers\AuthController.php:50
Stack trace:
#0 [internal function]: App\Http\Controllers\AuthController->login(Object(Laravel\Lumen\Http\Request))
#1 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\container\BoundMethod.php(32): call_user_func_array(Array, Array)
#2 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\container\Util.php(36): Illuminate\Container\BoundMethod::Illuminate\Container\{closure}()
#3 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\container\BoundMethod.php(90): Illuminate\Container\Util::unwrapIfClosure(Object(Closure))
#4 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\container\BoundMethod.php(34): Illuminate\Container\BoundMethod::callBoundMethod(Object(Laravel\Lumen\Application), Array, Object(Closure))
#5 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\container\Container.php(592): Illuminate\Container\BoundMethod::call(Object(Laravel\Lumen\Application), Array, Array, NULL)
#6 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(376): Illuminate\Container\Container->call(Array, Array)
#7 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(342): Laravel\Lumen\Application->callControllerCallable(Array, Array)
#8 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(316): Laravel\Lumen\Application->callLumenController(Object(App\Http\Controllers\AuthController), 'login', Array)
#9 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(278): Laravel\Lumen\Application->callControllerAction(Array)
#10 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(263): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#11 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(165): Laravel\Lumen\Application->handleFoundRoute(Array)
#12 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(416): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}(Object(Laravel\Lumen\Http\Request))
#13 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(171): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#14 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(108): Laravel\Lumen\Application->dispatch(NULL)
#15 C:\xampp\htdocs\lumenapijwt\public\index.php(28): Laravel\Lumen\Application->run()
#16 {main} {"exception":"[object] (Error(code: 0): Class 'App\\Http\\Controllers\\Auth' not found at C:\\xampp\\htdocs\\lumenapijwt\\app\\Http\\Controllers\\AuthController.php:50)
[stacktrace]
#0 [internal function]: App\\Http\\Controllers\\AuthController->login(Object(Laravel\\Lumen\\Http\\Request))
#1 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\container\\BoundMethod.php(32): call_user_func_array(Array, Array)
#2 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\container\\Util.php(36): Illuminate\\Container\\BoundMethod::Illuminate\\Container\\{closure}()
#3 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\container\\BoundMethod.php(90): Illuminate\\Container\\Util::unwrapIfClosure(Object(Closure))
#4 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\container\\BoundMethod.php(34): Illuminate\\Container\\BoundMethod::callBoundMethod(Object(Laravel\\Lumen\\Application), Array, Object(Closure))
#5 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\container\\Container.php(592): Illuminate\\Container\\BoundMethod::call(Object(Laravel\\Lumen\\Application), Array, Array, NULL)
#6 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(376): Illuminate\\Container\\Container->call(Array, Array)
#7 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(342): Laravel\\Lumen\\Application->callControllerCallable(Array, Array)
#8 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(316): Laravel\\Lumen\\Application->callLumenController(Object(App\\Http\\Controllers\\AuthController), 'login', Array)
#9 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(278): Laravel\\Lumen\\Application->callControllerAction(Array)
#10 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(263): Laravel\\Lumen\\Application->callActionOnArrayBasedRoute(Array)
#11 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(165): Laravel\\Lumen\\Application->handleFoundRoute(Array)
#12 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(416): Laravel\\Lumen\\Application->Laravel\\Lumen\\Concerns\\{closure}(Object(Laravel\\Lumen\\Http\\Request))
#13 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(171): Laravel\\Lumen\\Application->sendThroughPipeline(Array, Object(Closure))
#14 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(108): Laravel\\Lumen\\Application->dispatch(NULL)
#15 C:\\xampp\\htdocs\\lumenapijwt\\public\\index.php(28): Laravel\\Lumen\\Application->run()
#16 {main}
"} 
[2020-03-25 04:25:47] local.ERROR: BadMethodCallException: Call to undefined method App\User::middleware() in C:\xampp\htdocs\lumenapijwt\vendor\illuminate\support\Traits\ForwardsCalls.php:50
Stack trace:
#0 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\support\Traits\ForwardsCalls.php(36): Illuminate\Database\Eloquent\Model::throwBadMethodCallException('middleware')
#1 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\database\Eloquent\Model.php(1662): Illuminate\Database\Eloquent\Model->forwardCallTo(Object(Illuminate\Database\Eloquent\Builder), 'middleware', Array)
#2 C:\xampp\htdocs\lumenapijwt\app\User.php(66): Illuminate\Database\Eloquent\Model->__call('middleware', Array)
#3 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\auth\EloquentUserProvider.php(183): App\User->__construct()
#4 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\auth\EloquentUserProvider.php(170): Illuminate\Auth\EloquentUserProvider->createModel()
#5 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\auth\EloquentUserProvider.php(117): Illuminate\Auth\EloquentUserProvider->newModelQuery()
#6 C:\xampp\htdocs\lumenapijwt\vendor\tymon\jwt-auth\src\JWTGuard.php(124): Illuminate\Auth\EloquentUserProvider->retrieveByCredentials(Array)
#7 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\auth\AuthManager.php(307): Tymon\JWTAuth\JWTGuard->attempt(Array)
#8 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\support\Facades\Facade.php(261): Illuminate\Auth\AuthManager->__call('attempt', Array)
#9 C:\xampp\htdocs\lumenapijwt\app\Http\Controllers\AuthController.php(50): Illuminate\Support\Facades\Facade::__callStatic('attempt', Array)
#10 [internal function]: App\Http\Controllers\AuthController->login(Object(Laravel\Lumen\Http\Request))
#11 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\container\BoundMethod.php(32): call_user_func_array(Array, Array)
#12 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\container\Util.php(36): Illuminate\Container\BoundMethod::Illuminate\Container\{closure}()
#13 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\container\BoundMethod.php(90): Illuminate\Container\Util::unwrapIfClosure(Object(Closure))
#14 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\container\BoundMethod.php(34): Illuminate\Container\BoundMethod::callBoundMethod(Object(Laravel\Lumen\Application), Array, Object(Closure))
#15 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\container\Container.php(592): Illuminate\Container\BoundMethod::call(Object(Laravel\Lumen\Application), Array, Array, NULL)
#16 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(376): Illuminate\Container\Container->call(Array, Array)
#17 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(342): Laravel\Lumen\Application->callControllerCallable(Array, Array)
#18 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(316): Laravel\Lumen\Application->callLumenController(Object(App\Http\Controllers\AuthController), 'login', Array)
#19 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(278): Laravel\Lumen\Application->callControllerAction(Array)
#20 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(263): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#21 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(165): Laravel\Lumen\Application->handleFoundRoute(Array)
#22 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(416): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}(Object(Laravel\Lumen\Http\Request))
#23 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(171): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#24 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(108): Laravel\Lumen\Application->dispatch(NULL)
#25 C:\xampp\htdocs\lumenapijwt\public\index.php(28): Laravel\Lumen\Application->run()
#26 {main} {"exception":"[object] (BadMethodCallException(code: 0): Call to undefined method App\\User::middleware() at C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\support\\Traits\\ForwardsCalls.php:50)
[stacktrace]
#0 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\support\\Traits\\ForwardsCalls.php(36): Illuminate\\Database\\Eloquent\\Model::throwBadMethodCallException('middleware')
#1 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\database\\Eloquent\\Model.php(1662): Illuminate\\Database\\Eloquent\\Model->forwardCallTo(Object(Illuminate\\Database\\Eloquent\\Builder), 'middleware', Array)
#2 C:\\xampp\\htdocs\\lumenapijwt\\app\\User.php(66): Illuminate\\Database\\Eloquent\\Model->__call('middleware', Array)
#3 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\auth\\EloquentUserProvider.php(183): App\\User->__construct()
#4 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\auth\\EloquentUserProvider.php(170): Illuminate\\Auth\\EloquentUserProvider->createModel()
#5 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\auth\\EloquentUserProvider.php(117): Illuminate\\Auth\\EloquentUserProvider->newModelQuery()
#6 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\tymon\\jwt-auth\\src\\JWTGuard.php(124): Illuminate\\Auth\\EloquentUserProvider->retrieveByCredentials(Array)
#7 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\auth\\AuthManager.php(307): Tymon\\JWTAuth\\JWTGuard->attempt(Array)
#8 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\support\\Facades\\Facade.php(261): Illuminate\\Auth\\AuthManager->__call('attempt', Array)
#9 C:\\xampp\\htdocs\\lumenapijwt\\app\\Http\\Controllers\\AuthController.php(50): Illuminate\\Support\\Facades\\Facade::__callStatic('attempt', Array)
#10 [internal function]: App\\Http\\Controllers\\AuthController->login(Object(Laravel\\Lumen\\Http\\Request))
#11 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\container\\BoundMethod.php(32): call_user_func_array(Array, Array)
#12 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\container\\Util.php(36): Illuminate\\Container\\BoundMethod::Illuminate\\Container\\{closure}()
#13 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\container\\BoundMethod.php(90): Illuminate\\Container\\Util::unwrapIfClosure(Object(Closure))
#14 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\container\\BoundMethod.php(34): Illuminate\\Container\\BoundMethod::callBoundMethod(Object(Laravel\\Lumen\\Application), Array, Object(Closure))
#15 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\container\\Container.php(592): Illuminate\\Container\\BoundMethod::call(Object(Laravel\\Lumen\\Application), Array, Array, NULL)
#16 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(376): Illuminate\\Container\\Container->call(Array, Array)
#17 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(342): Laravel\\Lumen\\Application->callControllerCallable(Array, Array)
#18 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(316): Laravel\\Lumen\\Application->callLumenController(Object(App\\Http\\Controllers\\AuthController), 'login', Array)
#19 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(278): Laravel\\Lumen\\Application->callControllerAction(Array)
#20 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(263): Laravel\\Lumen\\Application->callActionOnArrayBasedRoute(Array)
#21 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(165): Laravel\\Lumen\\Application->handleFoundRoute(Array)
#22 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(416): Laravel\\Lumen\\Application->Laravel\\Lumen\\Concerns\\{closure}(Object(Laravel\\Lumen\\Http\\Request))
#23 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(171): Laravel\\Lumen\\Application->sendThroughPipeline(Array, Object(Closure))
#24 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(108): Laravel\\Lumen\\Application->dispatch(NULL)
#25 C:\\xampp\\htdocs\\lumenapijwt\\public\\index.php(28): Laravel\\Lumen\\Application->run()
#26 {main}
"} 
[2020-03-25 04:27:57] local.ERROR: BadMethodCallException: Call to undefined method App\User::middleware() in C:\xampp\htdocs\lumenapijwt\vendor\illuminate\support\Traits\ForwardsCalls.php:50
Stack trace:
#0 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\support\Traits\ForwardsCalls.php(36): Illuminate\Database\Eloquent\Model::throwBadMethodCallException('middleware')
#1 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\database\Eloquent\Model.php(1662): Illuminate\Database\Eloquent\Model->forwardCallTo(Object(Illuminate\Database\Eloquent\Builder), 'middleware', Array)
#2 C:\xampp\htdocs\lumenapijwt\app\User.php(66): Illuminate\Database\Eloquent\Model->__call('middleware', Array)
#3 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\auth\EloquentUserProvider.php(183): App\User->__construct()
#4 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\auth\EloquentUserProvider.php(170): Illuminate\Auth\EloquentUserProvider->createModel()
#5 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\auth\EloquentUserProvider.php(117): Illuminate\Auth\EloquentUserProvider->newModelQuery()
#6 C:\xampp\htdocs\lumenapijwt\vendor\tymon\jwt-auth\src\JWTGuard.php(124): Illuminate\Auth\EloquentUserProvider->retrieveByCredentials(Array)
#7 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\auth\AuthManager.php(307): Tymon\JWTAuth\JWTGuard->attempt(Array)
#8 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\support\Facades\Facade.php(261): Illuminate\Auth\AuthManager->__call('attempt', Array)
#9 C:\xampp\htdocs\lumenapijwt\app\Http\Controllers\AuthController.php(50): Illuminate\Support\Facades\Facade::__callStatic('attempt', Array)
#10 [internal function]: App\Http\Controllers\AuthController->login(Object(Laravel\Lumen\Http\Request))
#11 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\container\BoundMethod.php(32): call_user_func_array(Array, Array)
#12 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\container\Util.php(36): Illuminate\Container\BoundMethod::Illuminate\Container\{closure}()
#13 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\container\BoundMethod.php(90): Illuminate\Container\Util::unwrapIfClosure(Object(Closure))
#14 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\container\BoundMethod.php(34): Illuminate\Container\BoundMethod::callBoundMethod(Object(Laravel\Lumen\Application), Array, Object(Closure))
#15 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\container\Container.php(592): Illuminate\Container\BoundMethod::call(Object(Laravel\Lumen\Application), Array, Array, NULL)
#16 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(376): Illuminate\Container\Container->call(Array, Array)
#17 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(342): Laravel\Lumen\Application->callControllerCallable(Array, Array)
#18 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(316): Laravel\Lumen\Application->callLumenController(Object(App\Http\Controllers\AuthController), 'login', Array)
#19 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(278): Laravel\Lumen\Application->callControllerAction(Array)
#20 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(263): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#21 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(165): Laravel\Lumen\Application->handleFoundRoute(Array)
#22 [internal function]: Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}(Object(Laravel\Lumen\Http\Request))
#23 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Routing\Pipeline.php(48): call_user_func(Object(Closure), Object(Laravel\Lumen\Http\Request))
#24 C:\xampp\htdocs\lumenapijwt\app\Http\Middleware\ExampleMiddleware.php(18): Laravel\Lumen\Routing\Pipeline->Laravel\Lumen\Routing\{closure}(Object(Laravel\Lumen\Http\Request))
#25 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\pipeline\Pipeline.php(167): App\Http\Middleware\ExampleMiddleware->handle(Object(Laravel\Lumen\Http\Request), Object(Closure))
#26 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Laravel\Lumen\Http\Request))
#27 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Routing\Pipeline.php(30): call_user_func(Object(Closure), Object(Laravel\Lumen\Http\Request))
#28 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\pipeline\Pipeline.php(103): Laravel\Lumen\Routing\Pipeline->Laravel\Lumen\Routing\{closure}(Object(Laravel\Lumen\Http\Request))
#29 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(413): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#30 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(171): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#31 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(108): Laravel\Lumen\Application->dispatch(NULL)
#32 C:\xampp\htdocs\lumenapijwt\public\index.php(28): Laravel\Lumen\Application->run()
#33 {main} {"exception":"[object] (BadMethodCallException(code: 0): Call to undefined method App\\User::middleware() at C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\support\\Traits\\ForwardsCalls.php:50)
[stacktrace]
#0 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\support\\Traits\\ForwardsCalls.php(36): Illuminate\\Database\\Eloquent\\Model::throwBadMethodCallException('middleware')
#1 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\database\\Eloquent\\Model.php(1662): Illuminate\\Database\\Eloquent\\Model->forwardCallTo(Object(Illuminate\\Database\\Eloquent\\Builder), 'middleware', Array)
#2 C:\\xampp\\htdocs\\lumenapijwt\\app\\User.php(66): Illuminate\\Database\\Eloquent\\Model->__call('middleware', Array)
#3 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\auth\\EloquentUserProvider.php(183): App\\User->__construct()
#4 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\auth\\EloquentUserProvider.php(170): Illuminate\\Auth\\EloquentUserProvider->createModel()
#5 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\auth\\EloquentUserProvider.php(117): Illuminate\\Auth\\EloquentUserProvider->newModelQuery()
#6 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\tymon\\jwt-auth\\src\\JWTGuard.php(124): Illuminate\\Auth\\EloquentUserProvider->retrieveByCredentials(Array)
#7 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\auth\\AuthManager.php(307): Tymon\\JWTAuth\\JWTGuard->attempt(Array)
#8 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\support\\Facades\\Facade.php(261): Illuminate\\Auth\\AuthManager->__call('attempt', Array)
#9 C:\\xampp\\htdocs\\lumenapijwt\\app\\Http\\Controllers\\AuthController.php(50): Illuminate\\Support\\Facades\\Facade::__callStatic('attempt', Array)
#10 [internal function]: App\\Http\\Controllers\\AuthController->login(Object(Laravel\\Lumen\\Http\\Request))
#11 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\container\\BoundMethod.php(32): call_user_func_array(Array, Array)
#12 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\container\\Util.php(36): Illuminate\\Container\\BoundMethod::Illuminate\\Container\\{closure}()
#13 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\container\\BoundMethod.php(90): Illuminate\\Container\\Util::unwrapIfClosure(Object(Closure))
#14 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\container\\BoundMethod.php(34): Illuminate\\Container\\BoundMethod::callBoundMethod(Object(Laravel\\Lumen\\Application), Array, Object(Closure))
#15 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\container\\Container.php(592): Illuminate\\Container\\BoundMethod::call(Object(Laravel\\Lumen\\Application), Array, Array, NULL)
#16 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(376): Illuminate\\Container\\Container->call(Array, Array)
#17 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(342): Laravel\\Lumen\\Application->callControllerCallable(Array, Array)
#18 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(316): Laravel\\Lumen\\Application->callLumenController(Object(App\\Http\\Controllers\\AuthController), 'login', Array)
#19 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(278): Laravel\\Lumen\\Application->callControllerAction(Array)
#20 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(263): Laravel\\Lumen\\Application->callActionOnArrayBasedRoute(Array)
#21 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(165): Laravel\\Lumen\\Application->handleFoundRoute(Array)
#22 [internal function]: Laravel\\Lumen\\Application->Laravel\\Lumen\\Concerns\\{closure}(Object(Laravel\\Lumen\\Http\\Request))
#23 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Routing\\Pipeline.php(48): call_user_func(Object(Closure), Object(Laravel\\Lumen\\Http\\Request))
#24 C:\\xampp\\htdocs\\lumenapijwt\\app\\Http\\Middleware\\ExampleMiddleware.php(18): Laravel\\Lumen\\Routing\\Pipeline->Laravel\\Lumen\\Routing\\{closure}(Object(Laravel\\Lumen\\Http\\Request))
#25 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\pipeline\\Pipeline.php(167): App\\Http\\Middleware\\ExampleMiddleware->handle(Object(Laravel\\Lumen\\Http\\Request), Object(Closure))
#26 [internal function]: Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Object(Laravel\\Lumen\\Http\\Request))
#27 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Routing\\Pipeline.php(30): call_user_func(Object(Closure), Object(Laravel\\Lumen\\Http\\Request))
#28 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\pipeline\\Pipeline.php(103): Laravel\\Lumen\\Routing\\Pipeline->Laravel\\Lumen\\Routing\\{closure}(Object(Laravel\\Lumen\\Http\\Request))
#29 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(413): Illuminate\\Pipeline\\Pipeline->then(Object(Closure))
#30 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(171): Laravel\\Lumen\\Application->sendThroughPipeline(Array, Object(Closure))
#31 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(108): Laravel\\Lumen\\Application->dispatch(NULL)
#32 C:\\xampp\\htdocs\\lumenapijwt\\public\\index.php(28): Laravel\\Lumen\\Application->run()
#33 {main}
"} 
[2020-03-25 04:27:59] local.ERROR: BadMethodCallException: Call to undefined method App\User::middleware() in C:\xampp\htdocs\lumenapijwt\vendor\illuminate\support\Traits\ForwardsCalls.php:50
Stack trace:
#0 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\support\Traits\ForwardsCalls.php(36): Illuminate\Database\Eloquent\Model::throwBadMethodCallException('middleware')
#1 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\database\Eloquent\Model.php(1662): Illuminate\Database\Eloquent\Model->forwardCallTo(Object(Illuminate\Database\Eloquent\Builder), 'middleware', Array)
#2 C:\xampp\htdocs\lumenapijwt\app\User.php(66): Illuminate\Database\Eloquent\Model->__call('middleware', Array)
#3 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\auth\EloquentUserProvider.php(183): App\User->__construct()
#4 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\auth\EloquentUserProvider.php(170): Illuminate\Auth\EloquentUserProvider->createModel()
#5 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\auth\EloquentUserProvider.php(117): Illuminate\Auth\EloquentUserProvider->newModelQuery()
#6 C:\xampp\htdocs\lumenapijwt\vendor\tymon\jwt-auth\src\JWTGuard.php(124): Illuminate\Auth\EloquentUserProvider->retrieveByCredentials(Array)
#7 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\auth\AuthManager.php(307): Tymon\JWTAuth\JWTGuard->attempt(Array)
#8 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\support\Facades\Facade.php(261): Illuminate\Auth\AuthManager->__call('attempt', Array)
#9 C:\xampp\htdocs\lumenapijwt\app\Http\Controllers\AuthController.php(50): Illuminate\Support\Facades\Facade::__callStatic('attempt', Array)
#10 [internal function]: App\Http\Controllers\AuthController->login(Object(Laravel\Lumen\Http\Request))
#11 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\container\BoundMethod.php(32): call_user_func_array(Array, Array)
#12 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\container\Util.php(36): Illuminate\Container\BoundMethod::Illuminate\Container\{closure}()
#13 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\container\BoundMethod.php(90): Illuminate\Container\Util::unwrapIfClosure(Object(Closure))
#14 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\container\BoundMethod.php(34): Illuminate\Container\BoundMethod::callBoundMethod(Object(Laravel\Lumen\Application), Array, Object(Closure))
#15 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\container\Container.php(592): Illuminate\Container\BoundMethod::call(Object(Laravel\Lumen\Application), Array, Array, NULL)
#16 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(376): Illuminate\Container\Container->call(Array, Array)
#17 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(342): Laravel\Lumen\Application->callControllerCallable(Array, Array)
#18 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(316): Laravel\Lumen\Application->callLumenController(Object(App\Http\Controllers\AuthController), 'login', Array)
#19 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(278): Laravel\Lumen\Application->callControllerAction(Array)
#20 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(263): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#21 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(165): Laravel\Lumen\Application->handleFoundRoute(Array)
#22 [internal function]: Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}(Object(Laravel\Lumen\Http\Request))
#23 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Routing\Pipeline.php(48): call_user_func(Object(Closure), Object(Laravel\Lumen\Http\Request))
#24 C:\xampp\htdocs\lumenapijwt\app\Http\Middleware\ExampleMiddleware.php(18): Laravel\Lumen\Routing\Pipeline->Laravel\Lumen\Routing\{closure}(Object(Laravel\Lumen\Http\Request))
#25 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\pipeline\Pipeline.php(167): App\Http\Middleware\ExampleMiddleware->handle(Object(Laravel\Lumen\Http\Request), Object(Closure))
#26 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Laravel\Lumen\Http\Request))
#27 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Routing\Pipeline.php(30): call_user_func(Object(Closure), Object(Laravel\Lumen\Http\Request))
#28 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\pipeline\Pipeline.php(103): Laravel\Lumen\Routing\Pipeline->Laravel\Lumen\Routing\{closure}(Object(Laravel\Lumen\Http\Request))
#29 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(413): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#30 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(171): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#31 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(108): Laravel\Lumen\Application->dispatch(NULL)
#32 C:\xampp\htdocs\lumenapijwt\public\index.php(28): Laravel\Lumen\Application->run()
#33 {main} {"exception":"[object] (BadMethodCallException(code: 0): Call to undefined method App\\User::middleware() at C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\support\\Traits\\ForwardsCalls.php:50)
[stacktrace]
#0 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\support\\Traits\\ForwardsCalls.php(36): Illuminate\\Database\\Eloquent\\Model::throwBadMethodCallException('middleware')
#1 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\database\\Eloquent\\Model.php(1662): Illuminate\\Database\\Eloquent\\Model->forwardCallTo(Object(Illuminate\\Database\\Eloquent\\Builder), 'middleware', Array)
#2 C:\\xampp\\htdocs\\lumenapijwt\\app\\User.php(66): Illuminate\\Database\\Eloquent\\Model->__call('middleware', Array)
#3 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\auth\\EloquentUserProvider.php(183): App\\User->__construct()
#4 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\auth\\EloquentUserProvider.php(170): Illuminate\\Auth\\EloquentUserProvider->createModel()
#5 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\auth\\EloquentUserProvider.php(117): Illuminate\\Auth\\EloquentUserProvider->newModelQuery()
#6 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\tymon\\jwt-auth\\src\\JWTGuard.php(124): Illuminate\\Auth\\EloquentUserProvider->retrieveByCredentials(Array)
#7 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\auth\\AuthManager.php(307): Tymon\\JWTAuth\\JWTGuard->attempt(Array)
#8 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\support\\Facades\\Facade.php(261): Illuminate\\Auth\\AuthManager->__call('attempt', Array)
#9 C:\\xampp\\htdocs\\lumenapijwt\\app\\Http\\Controllers\\AuthController.php(50): Illuminate\\Support\\Facades\\Facade::__callStatic('attempt', Array)
#10 [internal function]: App\\Http\\Controllers\\AuthController->login(Object(Laravel\\Lumen\\Http\\Request))
#11 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\container\\BoundMethod.php(32): call_user_func_array(Array, Array)
#12 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\container\\Util.php(36): Illuminate\\Container\\BoundMethod::Illuminate\\Container\\{closure}()
#13 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\container\\BoundMethod.php(90): Illuminate\\Container\\Util::unwrapIfClosure(Object(Closure))
#14 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\container\\BoundMethod.php(34): Illuminate\\Container\\BoundMethod::callBoundMethod(Object(Laravel\\Lumen\\Application), Array, Object(Closure))
#15 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\container\\Container.php(592): Illuminate\\Container\\BoundMethod::call(Object(Laravel\\Lumen\\Application), Array, Array, NULL)
#16 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(376): Illuminate\\Container\\Container->call(Array, Array)
#17 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(342): Laravel\\Lumen\\Application->callControllerCallable(Array, Array)
#18 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(316): Laravel\\Lumen\\Application->callLumenController(Object(App\\Http\\Controllers\\AuthController), 'login', Array)
#19 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(278): Laravel\\Lumen\\Application->callControllerAction(Array)
#20 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(263): Laravel\\Lumen\\Application->callActionOnArrayBasedRoute(Array)
#21 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(165): Laravel\\Lumen\\Application->handleFoundRoute(Array)
#22 [internal function]: Laravel\\Lumen\\Application->Laravel\\Lumen\\Concerns\\{closure}(Object(Laravel\\Lumen\\Http\\Request))
#23 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Routing\\Pipeline.php(48): call_user_func(Object(Closure), Object(Laravel\\Lumen\\Http\\Request))
#24 C:\\xampp\\htdocs\\lumenapijwt\\app\\Http\\Middleware\\ExampleMiddleware.php(18): Laravel\\Lumen\\Routing\\Pipeline->Laravel\\Lumen\\Routing\\{closure}(Object(Laravel\\Lumen\\Http\\Request))
#25 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\pipeline\\Pipeline.php(167): App\\Http\\Middleware\\ExampleMiddleware->handle(Object(Laravel\\Lumen\\Http\\Request), Object(Closure))
#26 [internal function]: Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Object(Laravel\\Lumen\\Http\\Request))
#27 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Routing\\Pipeline.php(30): call_user_func(Object(Closure), Object(Laravel\\Lumen\\Http\\Request))
#28 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\pipeline\\Pipeline.php(103): Laravel\\Lumen\\Routing\\Pipeline->Laravel\\Lumen\\Routing\\{closure}(Object(Laravel\\Lumen\\Http\\Request))
#29 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(413): Illuminate\\Pipeline\\Pipeline->then(Object(Closure))
#30 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(171): Laravel\\Lumen\\Application->sendThroughPipeline(Array, Object(Closure))
#31 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(108): Laravel\\Lumen\\Application->dispatch(NULL)
#32 C:\\xampp\\htdocs\\lumenapijwt\\public\\index.php(28): Laravel\\Lumen\\Application->run()
#33 {main}
"} 
[2020-03-25 04:28:11] local.ERROR: BadMethodCallException: Call to undefined method App\User::middleware() in C:\xampp\htdocs\lumenapijwt\vendor\illuminate\support\Traits\ForwardsCalls.php:50
Stack trace:
#0 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\support\Traits\ForwardsCalls.php(36): Illuminate\Database\Eloquent\Model::throwBadMethodCallException('middleware')
#1 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\database\Eloquent\Model.php(1662): Illuminate\Database\Eloquent\Model->forwardCallTo(Object(Illuminate\Database\Eloquent\Builder), 'middleware', Array)
#2 C:\xampp\htdocs\lumenapijwt\app\User.php(66): Illuminate\Database\Eloquent\Model->__call('middleware', Array)
#3 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\auth\EloquentUserProvider.php(183): App\User->__construct()
#4 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\auth\EloquentUserProvider.php(170): Illuminate\Auth\EloquentUserProvider->createModel()
#5 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\auth\EloquentUserProvider.php(117): Illuminate\Auth\EloquentUserProvider->newModelQuery()
#6 C:\xampp\htdocs\lumenapijwt\vendor\tymon\jwt-auth\src\JWTGuard.php(124): Illuminate\Auth\EloquentUserProvider->retrieveByCredentials(Array)
#7 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\auth\AuthManager.php(307): Tymon\JWTAuth\JWTGuard->attempt(Array)
#8 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\support\Facades\Facade.php(261): Illuminate\Auth\AuthManager->__call('attempt', Array)
#9 C:\xampp\htdocs\lumenapijwt\app\Http\Controllers\AuthController.php(50): Illuminate\Support\Facades\Facade::__callStatic('attempt', Array)
#10 [internal function]: App\Http\Controllers\AuthController->login(Object(Laravel\Lumen\Http\Request))
#11 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\container\BoundMethod.php(32): call_user_func_array(Array, Array)
#12 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\container\Util.php(36): Illuminate\Container\BoundMethod::Illuminate\Container\{closure}()
#13 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\container\BoundMethod.php(90): Illuminate\Container\Util::unwrapIfClosure(Object(Closure))
#14 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\container\BoundMethod.php(34): Illuminate\Container\BoundMethod::callBoundMethod(Object(Laravel\Lumen\Application), Array, Object(Closure))
#15 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\container\Container.php(592): Illuminate\Container\BoundMethod::call(Object(Laravel\Lumen\Application), Array, Array, NULL)
#16 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(376): Illuminate\Container\Container->call(Array, Array)
#17 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(342): Laravel\Lumen\Application->callControllerCallable(Array, Array)
#18 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(316): Laravel\Lumen\Application->callLumenController(Object(App\Http\Controllers\AuthController), 'login', Array)
#19 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(278): Laravel\Lumen\Application->callControllerAction(Array)
#20 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(263): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#21 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(165): Laravel\Lumen\Application->handleFoundRoute(Array)
#22 [internal function]: Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}(Object(Laravel\Lumen\Http\Request))
#23 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Routing\Pipeline.php(48): call_user_func(Object(Closure), Object(Laravel\Lumen\Http\Request))
#24 C:\xampp\htdocs\lumenapijwt\app\Http\Middleware\ExampleMiddleware.php(18): Laravel\Lumen\Routing\Pipeline->Laravel\Lumen\Routing\{closure}(Object(Laravel\Lumen\Http\Request))
#25 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\pipeline\Pipeline.php(167): App\Http\Middleware\ExampleMiddleware->handle(Object(Laravel\Lumen\Http\Request), Object(Closure))
#26 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Laravel\Lumen\Http\Request))
#27 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Routing\Pipeline.php(30): call_user_func(Object(Closure), Object(Laravel\Lumen\Http\Request))
#28 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\pipeline\Pipeline.php(103): Laravel\Lumen\Routing\Pipeline->Laravel\Lumen\Routing\{closure}(Object(Laravel\Lumen\Http\Request))
#29 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(413): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#30 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(171): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#31 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(108): Laravel\Lumen\Application->dispatch(NULL)
#32 C:\xampp\htdocs\lumenapijwt\public\index.php(28): Laravel\Lumen\Application->run()
#33 {main} {"exception":"[object] (BadMethodCallException(code: 0): Call to undefined method App\\User::middleware() at C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\support\\Traits\\ForwardsCalls.php:50)
[stacktrace]
#0 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\support\\Traits\\ForwardsCalls.php(36): Illuminate\\Database\\Eloquent\\Model::throwBadMethodCallException('middleware')
#1 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\database\\Eloquent\\Model.php(1662): Illuminate\\Database\\Eloquent\\Model->forwardCallTo(Object(Illuminate\\Database\\Eloquent\\Builder), 'middleware', Array)
#2 C:\\xampp\\htdocs\\lumenapijwt\\app\\User.php(66): Illuminate\\Database\\Eloquent\\Model->__call('middleware', Array)
#3 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\auth\\EloquentUserProvider.php(183): App\\User->__construct()
#4 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\auth\\EloquentUserProvider.php(170): Illuminate\\Auth\\EloquentUserProvider->createModel()
#5 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\auth\\EloquentUserProvider.php(117): Illuminate\\Auth\\EloquentUserProvider->newModelQuery()
#6 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\tymon\\jwt-auth\\src\\JWTGuard.php(124): Illuminate\\Auth\\EloquentUserProvider->retrieveByCredentials(Array)
#7 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\auth\\AuthManager.php(307): Tymon\\JWTAuth\\JWTGuard->attempt(Array)
#8 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\support\\Facades\\Facade.php(261): Illuminate\\Auth\\AuthManager->__call('attempt', Array)
#9 C:\\xampp\\htdocs\\lumenapijwt\\app\\Http\\Controllers\\AuthController.php(50): Illuminate\\Support\\Facades\\Facade::__callStatic('attempt', Array)
#10 [internal function]: App\\Http\\Controllers\\AuthController->login(Object(Laravel\\Lumen\\Http\\Request))
#11 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\container\\BoundMethod.php(32): call_user_func_array(Array, Array)
#12 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\container\\Util.php(36): Illuminate\\Container\\BoundMethod::Illuminate\\Container\\{closure}()
#13 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\container\\BoundMethod.php(90): Illuminate\\Container\\Util::unwrapIfClosure(Object(Closure))
#14 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\container\\BoundMethod.php(34): Illuminate\\Container\\BoundMethod::callBoundMethod(Object(Laravel\\Lumen\\Application), Array, Object(Closure))
#15 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\container\\Container.php(592): Illuminate\\Container\\BoundMethod::call(Object(Laravel\\Lumen\\Application), Array, Array, NULL)
#16 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(376): Illuminate\\Container\\Container->call(Array, Array)
#17 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(342): Laravel\\Lumen\\Application->callControllerCallable(Array, Array)
#18 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(316): Laravel\\Lumen\\Application->callLumenController(Object(App\\Http\\Controllers\\AuthController), 'login', Array)
#19 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(278): Laravel\\Lumen\\Application->callControllerAction(Array)
#20 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(263): Laravel\\Lumen\\Application->callActionOnArrayBasedRoute(Array)
#21 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(165): Laravel\\Lumen\\Application->handleFoundRoute(Array)
#22 [internal function]: Laravel\\Lumen\\Application->Laravel\\Lumen\\Concerns\\{closure}(Object(Laravel\\Lumen\\Http\\Request))
#23 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Routing\\Pipeline.php(48): call_user_func(Object(Closure), Object(Laravel\\Lumen\\Http\\Request))
#24 C:\\xampp\\htdocs\\lumenapijwt\\app\\Http\\Middleware\\ExampleMiddleware.php(18): Laravel\\Lumen\\Routing\\Pipeline->Laravel\\Lumen\\Routing\\{closure}(Object(Laravel\\Lumen\\Http\\Request))
#25 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\pipeline\\Pipeline.php(167): App\\Http\\Middleware\\ExampleMiddleware->handle(Object(Laravel\\Lumen\\Http\\Request), Object(Closure))
#26 [internal function]: Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Object(Laravel\\Lumen\\Http\\Request))
#27 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Routing\\Pipeline.php(30): call_user_func(Object(Closure), Object(Laravel\\Lumen\\Http\\Request))
#28 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\pipeline\\Pipeline.php(103): Laravel\\Lumen\\Routing\\Pipeline->Laravel\\Lumen\\Routing\\{closure}(Object(Laravel\\Lumen\\Http\\Request))
#29 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(413): Illuminate\\Pipeline\\Pipeline->then(Object(Closure))
#30 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(171): Laravel\\Lumen\\Application->sendThroughPipeline(Array, Object(Closure))
#31 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(108): Laravel\\Lumen\\Application->dispatch(NULL)
#32 C:\\xampp\\htdocs\\lumenapijwt\\public\\index.php(28): Laravel\\Lumen\\Application->run()
#33 {main}
"} 
[2020-03-25 04:30:02] local.ERROR: BadMethodCallException: Call to undefined method App\User::middleware() in C:\xampp\htdocs\lumenapijwt\vendor\illuminate\support\Traits\ForwardsCalls.php:50
Stack trace:
#0 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\support\Traits\ForwardsCalls.php(36): Illuminate\Database\Eloquent\Model::throwBadMethodCallException('middleware')
#1 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\database\Eloquent\Model.php(1662): Illuminate\Database\Eloquent\Model->forwardCallTo(Object(Illuminate\Database\Eloquent\Builder), 'middleware', Array)
#2 C:\xampp\htdocs\lumenapijwt\app\User.php(66): Illuminate\Database\Eloquent\Model->__call('middleware', Array)
#3 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\auth\EloquentUserProvider.php(183): App\User->__construct()
#4 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\auth\EloquentUserProvider.php(170): Illuminate\Auth\EloquentUserProvider->createModel()
#5 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\auth\EloquentUserProvider.php(117): Illuminate\Auth\EloquentUserProvider->newModelQuery()
#6 C:\xampp\htdocs\lumenapijwt\vendor\tymon\jwt-auth\src\JWTGuard.php(124): Illuminate\Auth\EloquentUserProvider->retrieveByCredentials(Array)
#7 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\auth\AuthManager.php(307): Tymon\JWTAuth\JWTGuard->attempt(Array)
#8 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\support\Facades\Facade.php(261): Illuminate\Auth\AuthManager->__call('attempt', Array)
#9 C:\xampp\htdocs\lumenapijwt\app\Http\Controllers\AuthController.php(50): Illuminate\Support\Facades\Facade::__callStatic('attempt', Array)
#10 [internal function]: App\Http\Controllers\AuthController->login(Object(Laravel\Lumen\Http\Request))
#11 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\container\BoundMethod.php(32): call_user_func_array(Array, Array)
#12 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\container\Util.php(36): Illuminate\Container\BoundMethod::Illuminate\Container\{closure}()
#13 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\container\BoundMethod.php(90): Illuminate\Container\Util::unwrapIfClosure(Object(Closure))
#14 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\container\BoundMethod.php(34): Illuminate\Container\BoundMethod::callBoundMethod(Object(Laravel\Lumen\Application), Array, Object(Closure))
#15 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\container\Container.php(592): Illuminate\Container\BoundMethod::call(Object(Laravel\Lumen\Application), Array, Array, NULL)
#16 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(376): Illuminate\Container\Container->call(Array, Array)
#17 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(342): Laravel\Lumen\Application->callControllerCallable(Array, Array)
#18 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(316): Laravel\Lumen\Application->callLumenController(Object(App\Http\Controllers\AuthController), 'login', Array)
#19 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(278): Laravel\Lumen\Application->callControllerAction(Array)
#20 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(263): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#21 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(165): Laravel\Lumen\Application->handleFoundRoute(Array)
#22 [internal function]: Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}(Object(Laravel\Lumen\Http\Request))
#23 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Routing\Pipeline.php(48): call_user_func(Object(Closure), Object(Laravel\Lumen\Http\Request))
#24 C:\xampp\htdocs\lumenapijwt\app\Http\Middleware\ExampleMiddleware.php(18): Laravel\Lumen\Routing\Pipeline->Laravel\Lumen\Routing\{closure}(Object(Laravel\Lumen\Http\Request))
#25 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\pipeline\Pipeline.php(167): App\Http\Middleware\ExampleMiddleware->handle(Object(Laravel\Lumen\Http\Request), Object(Closure))
#26 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Laravel\Lumen\Http\Request))
#27 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Routing\Pipeline.php(30): call_user_func(Object(Closure), Object(Laravel\Lumen\Http\Request))
#28 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\pipeline\Pipeline.php(103): Laravel\Lumen\Routing\Pipeline->Laravel\Lumen\Routing\{closure}(Object(Laravel\Lumen\Http\Request))
#29 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(413): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#30 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(171): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#31 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(108): Laravel\Lumen\Application->dispatch(NULL)
#32 C:\xampp\htdocs\lumenapijwt\public\index.php(28): Laravel\Lumen\Application->run()
#33 {main} {"exception":"[object] (BadMethodCallException(code: 0): Call to undefined method App\\User::middleware() at C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\support\\Traits\\ForwardsCalls.php:50)
[stacktrace]
#0 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\support\\Traits\\ForwardsCalls.php(36): Illuminate\\Database\\Eloquent\\Model::throwBadMethodCallException('middleware')
#1 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\database\\Eloquent\\Model.php(1662): Illuminate\\Database\\Eloquent\\Model->forwardCallTo(Object(Illuminate\\Database\\Eloquent\\Builder), 'middleware', Array)
#2 C:\\xampp\\htdocs\\lumenapijwt\\app\\User.php(66): Illuminate\\Database\\Eloquent\\Model->__call('middleware', Array)
#3 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\auth\\EloquentUserProvider.php(183): App\\User->__construct()
#4 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\auth\\EloquentUserProvider.php(170): Illuminate\\Auth\\EloquentUserProvider->createModel()
#5 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\auth\\EloquentUserProvider.php(117): Illuminate\\Auth\\EloquentUserProvider->newModelQuery()
#6 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\tymon\\jwt-auth\\src\\JWTGuard.php(124): Illuminate\\Auth\\EloquentUserProvider->retrieveByCredentials(Array)
#7 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\auth\\AuthManager.php(307): Tymon\\JWTAuth\\JWTGuard->attempt(Array)
#8 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\support\\Facades\\Facade.php(261): Illuminate\\Auth\\AuthManager->__call('attempt', Array)
#9 C:\\xampp\\htdocs\\lumenapijwt\\app\\Http\\Controllers\\AuthController.php(50): Illuminate\\Support\\Facades\\Facade::__callStatic('attempt', Array)
#10 [internal function]: App\\Http\\Controllers\\AuthController->login(Object(Laravel\\Lumen\\Http\\Request))
#11 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\container\\BoundMethod.php(32): call_user_func_array(Array, Array)
#12 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\container\\Util.php(36): Illuminate\\Container\\BoundMethod::Illuminate\\Container\\{closure}()
#13 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\container\\BoundMethod.php(90): Illuminate\\Container\\Util::unwrapIfClosure(Object(Closure))
#14 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\container\\BoundMethod.php(34): Illuminate\\Container\\BoundMethod::callBoundMethod(Object(Laravel\\Lumen\\Application), Array, Object(Closure))
#15 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\container\\Container.php(592): Illuminate\\Container\\BoundMethod::call(Object(Laravel\\Lumen\\Application), Array, Array, NULL)
#16 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(376): Illuminate\\Container\\Container->call(Array, Array)
#17 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(342): Laravel\\Lumen\\Application->callControllerCallable(Array, Array)
#18 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(316): Laravel\\Lumen\\Application->callLumenController(Object(App\\Http\\Controllers\\AuthController), 'login', Array)
#19 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(278): Laravel\\Lumen\\Application->callControllerAction(Array)
#20 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(263): Laravel\\Lumen\\Application->callActionOnArrayBasedRoute(Array)
#21 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(165): Laravel\\Lumen\\Application->handleFoundRoute(Array)
#22 [internal function]: Laravel\\Lumen\\Application->Laravel\\Lumen\\Concerns\\{closure}(Object(Laravel\\Lumen\\Http\\Request))
#23 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Routing\\Pipeline.php(48): call_user_func(Object(Closure), Object(Laravel\\Lumen\\Http\\Request))
#24 C:\\xampp\\htdocs\\lumenapijwt\\app\\Http\\Middleware\\ExampleMiddleware.php(18): Laravel\\Lumen\\Routing\\Pipeline->Laravel\\Lumen\\Routing\\{closure}(Object(Laravel\\Lumen\\Http\\Request))
#25 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\pipeline\\Pipeline.php(167): App\\Http\\Middleware\\ExampleMiddleware->handle(Object(Laravel\\Lumen\\Http\\Request), Object(Closure))
#26 [internal function]: Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Object(Laravel\\Lumen\\Http\\Request))
#27 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Routing\\Pipeline.php(30): call_user_func(Object(Closure), Object(Laravel\\Lumen\\Http\\Request))
#28 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\pipeline\\Pipeline.php(103): Laravel\\Lumen\\Routing\\Pipeline->Laravel\\Lumen\\Routing\\{closure}(Object(Laravel\\Lumen\\Http\\Request))
#29 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(413): Illuminate\\Pipeline\\Pipeline->then(Object(Closure))
#30 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(171): Laravel\\Lumen\\Application->sendThroughPipeline(Array, Object(Closure))
#31 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(108): Laravel\\Lumen\\Application->dispatch(NULL)
#32 C:\\xampp\\htdocs\\lumenapijwt\\public\\index.php(28): Laravel\\Lumen\\Application->run()
#33 {main}
"} 
[2020-03-25 04:30:04] local.ERROR: BadMethodCallException: Call to undefined method App\User::middleware() in C:\xampp\htdocs\lumenapijwt\vendor\illuminate\support\Traits\ForwardsCalls.php:50
Stack trace:
#0 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\support\Traits\ForwardsCalls.php(36): Illuminate\Database\Eloquent\Model::throwBadMethodCallException('middleware')
#1 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\database\Eloquent\Model.php(1662): Illuminate\Database\Eloquent\Model->forwardCallTo(Object(Illuminate\Database\Eloquent\Builder), 'middleware', Array)
#2 C:\xampp\htdocs\lumenapijwt\app\User.php(66): Illuminate\Database\Eloquent\Model->__call('middleware', Array)
#3 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\auth\EloquentUserProvider.php(183): App\User->__construct()
#4 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\auth\EloquentUserProvider.php(170): Illuminate\Auth\EloquentUserProvider->createModel()
#5 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\auth\EloquentUserProvider.php(117): Illuminate\Auth\EloquentUserProvider->newModelQuery()
#6 C:\xampp\htdocs\lumenapijwt\vendor\tymon\jwt-auth\src\JWTGuard.php(124): Illuminate\Auth\EloquentUserProvider->retrieveByCredentials(Array)
#7 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\auth\AuthManager.php(307): Tymon\JWTAuth\JWTGuard->attempt(Array)
#8 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\support\Facades\Facade.php(261): Illuminate\Auth\AuthManager->__call('attempt', Array)
#9 C:\xampp\htdocs\lumenapijwt\app\Http\Controllers\AuthController.php(50): Illuminate\Support\Facades\Facade::__callStatic('attempt', Array)
#10 [internal function]: App\Http\Controllers\AuthController->login(Object(Laravel\Lumen\Http\Request))
#11 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\container\BoundMethod.php(32): call_user_func_array(Array, Array)
#12 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\container\Util.php(36): Illuminate\Container\BoundMethod::Illuminate\Container\{closure}()
#13 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\container\BoundMethod.php(90): Illuminate\Container\Util::unwrapIfClosure(Object(Closure))
#14 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\container\BoundMethod.php(34): Illuminate\Container\BoundMethod::callBoundMethod(Object(Laravel\Lumen\Application), Array, Object(Closure))
#15 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\container\Container.php(592): Illuminate\Container\BoundMethod::call(Object(Laravel\Lumen\Application), Array, Array, NULL)
#16 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(376): Illuminate\Container\Container->call(Array, Array)
#17 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(342): Laravel\Lumen\Application->callControllerCallable(Array, Array)
#18 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(316): Laravel\Lumen\Application->callLumenController(Object(App\Http\Controllers\AuthController), 'login', Array)
#19 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(278): Laravel\Lumen\Application->callControllerAction(Array)
#20 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(263): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#21 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(165): Laravel\Lumen\Application->handleFoundRoute(Array)
#22 [internal function]: Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}(Object(Laravel\Lumen\Http\Request))
#23 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Routing\Pipeline.php(48): call_user_func(Object(Closure), Object(Laravel\Lumen\Http\Request))
#24 C:\xampp\htdocs\lumenapijwt\app\Http\Middleware\ExampleMiddleware.php(18): Laravel\Lumen\Routing\Pipeline->Laravel\Lumen\Routing\{closure}(Object(Laravel\Lumen\Http\Request))
#25 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\pipeline\Pipeline.php(167): App\Http\Middleware\ExampleMiddleware->handle(Object(Laravel\Lumen\Http\Request), Object(Closure))
#26 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Laravel\Lumen\Http\Request))
#27 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Routing\Pipeline.php(30): call_user_func(Object(Closure), Object(Laravel\Lumen\Http\Request))
#28 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\pipeline\Pipeline.php(103): Laravel\Lumen\Routing\Pipeline->Laravel\Lumen\Routing\{closure}(Object(Laravel\Lumen\Http\Request))
#29 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(413): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#30 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(171): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#31 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(108): Laravel\Lumen\Application->dispatch(NULL)
#32 C:\xampp\htdocs\lumenapijwt\public\index.php(28): Laravel\Lumen\Application->run()
#33 {main} {"exception":"[object] (BadMethodCallException(code: 0): Call to undefined method App\\User::middleware() at C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\support\\Traits\\ForwardsCalls.php:50)
[stacktrace]
#0 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\support\\Traits\\ForwardsCalls.php(36): Illuminate\\Database\\Eloquent\\Model::throwBadMethodCallException('middleware')
#1 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\database\\Eloquent\\Model.php(1662): Illuminate\\Database\\Eloquent\\Model->forwardCallTo(Object(Illuminate\\Database\\Eloquent\\Builder), 'middleware', Array)
#2 C:\\xampp\\htdocs\\lumenapijwt\\app\\User.php(66): Illuminate\\Database\\Eloquent\\Model->__call('middleware', Array)
#3 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\auth\\EloquentUserProvider.php(183): App\\User->__construct()
#4 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\auth\\EloquentUserProvider.php(170): Illuminate\\Auth\\EloquentUserProvider->createModel()
#5 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\auth\\EloquentUserProvider.php(117): Illuminate\\Auth\\EloquentUserProvider->newModelQuery()
#6 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\tymon\\jwt-auth\\src\\JWTGuard.php(124): Illuminate\\Auth\\EloquentUserProvider->retrieveByCredentials(Array)
#7 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\auth\\AuthManager.php(307): Tymon\\JWTAuth\\JWTGuard->attempt(Array)
#8 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\support\\Facades\\Facade.php(261): Illuminate\\Auth\\AuthManager->__call('attempt', Array)
#9 C:\\xampp\\htdocs\\lumenapijwt\\app\\Http\\Controllers\\AuthController.php(50): Illuminate\\Support\\Facades\\Facade::__callStatic('attempt', Array)
#10 [internal function]: App\\Http\\Controllers\\AuthController->login(Object(Laravel\\Lumen\\Http\\Request))
#11 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\container\\BoundMethod.php(32): call_user_func_array(Array, Array)
#12 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\container\\Util.php(36): Illuminate\\Container\\BoundMethod::Illuminate\\Container\\{closure}()
#13 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\container\\BoundMethod.php(90): Illuminate\\Container\\Util::unwrapIfClosure(Object(Closure))
#14 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\container\\BoundMethod.php(34): Illuminate\\Container\\BoundMethod::callBoundMethod(Object(Laravel\\Lumen\\Application), Array, Object(Closure))
#15 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\container\\Container.php(592): Illuminate\\Container\\BoundMethod::call(Object(Laravel\\Lumen\\Application), Array, Array, NULL)
#16 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(376): Illuminate\\Container\\Container->call(Array, Array)
#17 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(342): Laravel\\Lumen\\Application->callControllerCallable(Array, Array)
#18 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(316): Laravel\\Lumen\\Application->callLumenController(Object(App\\Http\\Controllers\\AuthController), 'login', Array)
#19 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(278): Laravel\\Lumen\\Application->callControllerAction(Array)
#20 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(263): Laravel\\Lumen\\Application->callActionOnArrayBasedRoute(Array)
#21 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(165): Laravel\\Lumen\\Application->handleFoundRoute(Array)
#22 [internal function]: Laravel\\Lumen\\Application->Laravel\\Lumen\\Concerns\\{closure}(Object(Laravel\\Lumen\\Http\\Request))
#23 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Routing\\Pipeline.php(48): call_user_func(Object(Closure), Object(Laravel\\Lumen\\Http\\Request))
#24 C:\\xampp\\htdocs\\lumenapijwt\\app\\Http\\Middleware\\ExampleMiddleware.php(18): Laravel\\Lumen\\Routing\\Pipeline->Laravel\\Lumen\\Routing\\{closure}(Object(Laravel\\Lumen\\Http\\Request))
#25 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\pipeline\\Pipeline.php(167): App\\Http\\Middleware\\ExampleMiddleware->handle(Object(Laravel\\Lumen\\Http\\Request), Object(Closure))
#26 [internal function]: Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Object(Laravel\\Lumen\\Http\\Request))
#27 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Routing\\Pipeline.php(30): call_user_func(Object(Closure), Object(Laravel\\Lumen\\Http\\Request))
#28 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\pipeline\\Pipeline.php(103): Laravel\\Lumen\\Routing\\Pipeline->Laravel\\Lumen\\Routing\\{closure}(Object(Laravel\\Lumen\\Http\\Request))
#29 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(413): Illuminate\\Pipeline\\Pipeline->then(Object(Closure))
#30 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(171): Laravel\\Lumen\\Application->sendThroughPipeline(Array, Object(Closure))
#31 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(108): Laravel\\Lumen\\Application->dispatch(NULL)
#32 C:\\xampp\\htdocs\\lumenapijwt\\public\\index.php(28): Laravel\\Lumen\\Application->run()
#33 {main}
"} 
[2020-03-25 04:33:56] local.ERROR: BadMethodCallException: Call to undefined method App\User::middleware() in C:\xampp\htdocs\lumenapijwt\vendor\illuminate\support\Traits\ForwardsCalls.php:50
Stack trace:
#0 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\support\Traits\ForwardsCalls.php(36): Illuminate\Database\Eloquent\Model::throwBadMethodCallException('middleware')
#1 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\database\Eloquent\Model.php(1662): Illuminate\Database\Eloquent\Model->forwardCallTo(Object(Illuminate\Database\Eloquent\Builder), 'middleware', Array)
#2 C:\xampp\htdocs\lumenapijwt\app\User.php(66): Illuminate\Database\Eloquent\Model->__call('middleware', Array)
#3 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\auth\EloquentUserProvider.php(183): App\User->__construct()
#4 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\auth\EloquentUserProvider.php(170): Illuminate\Auth\EloquentUserProvider->createModel()
#5 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\auth\EloquentUserProvider.php(117): Illuminate\Auth\EloquentUserProvider->newModelQuery()
#6 C:\xampp\htdocs\lumenapijwt\vendor\tymon\jwt-auth\src\JWTGuard.php(124): Illuminate\Auth\EloquentUserProvider->retrieveByCredentials(Array)
#7 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\auth\AuthManager.php(307): Tymon\JWTAuth\JWTGuard->attempt(Array)
#8 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\support\Facades\Facade.php(261): Illuminate\Auth\AuthManager->__call('attempt', Array)
#9 C:\xampp\htdocs\lumenapijwt\app\Http\Controllers\AuthController.php(50): Illuminate\Support\Facades\Facade::__callStatic('attempt', Array)
#10 [internal function]: App\Http\Controllers\AuthController->login(Object(Laravel\Lumen\Http\Request))
#11 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\container\BoundMethod.php(32): call_user_func_array(Array, Array)
#12 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\container\Util.php(36): Illuminate\Container\BoundMethod::Illuminate\Container\{closure}()
#13 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\container\BoundMethod.php(90): Illuminate\Container\Util::unwrapIfClosure(Object(Closure))
#14 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\container\BoundMethod.php(34): Illuminate\Container\BoundMethod::callBoundMethod(Object(Laravel\Lumen\Application), Array, Object(Closure))
#15 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\container\Container.php(592): Illuminate\Container\BoundMethod::call(Object(Laravel\Lumen\Application), Array, Array, NULL)
#16 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(376): Illuminate\Container\Container->call(Array, Array)
#17 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(342): Laravel\Lumen\Application->callControllerCallable(Array, Array)
#18 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(316): Laravel\Lumen\Application->callLumenController(Object(App\Http\Controllers\AuthController), 'login', Array)
#19 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(278): Laravel\Lumen\Application->callControllerAction(Array)
#20 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(263): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#21 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(165): Laravel\Lumen\Application->handleFoundRoute(Array)
#22 [internal function]: Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}(Object(Laravel\Lumen\Http\Request))
#23 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Routing\Pipeline.php(48): call_user_func(Object(Closure), Object(Laravel\Lumen\Http\Request))
#24 C:\xampp\htdocs\lumenapijwt\app\Http\Middleware\ExampleMiddleware.php(18): Laravel\Lumen\Routing\Pipeline->Laravel\Lumen\Routing\{closure}(Object(Laravel\Lumen\Http\Request))
#25 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\pipeline\Pipeline.php(167): App\Http\Middleware\ExampleMiddleware->handle(Object(Laravel\Lumen\Http\Request), Object(Closure))
#26 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Laravel\Lumen\Http\Request))
#27 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Routing\Pipeline.php(30): call_user_func(Object(Closure), Object(Laravel\Lumen\Http\Request))
#28 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\pipeline\Pipeline.php(103): Laravel\Lumen\Routing\Pipeline->Laravel\Lumen\Routing\{closure}(Object(Laravel\Lumen\Http\Request))
#29 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(413): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#30 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(171): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#31 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(108): Laravel\Lumen\Application->dispatch(NULL)
#32 C:\xampp\htdocs\lumenapijwt\public\index.php(28): Laravel\Lumen\Application->run()
#33 {main} {"exception":"[object] (BadMethodCallException(code: 0): Call to undefined method App\\User::middleware() at C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\support\\Traits\\ForwardsCalls.php:50)
[stacktrace]
#0 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\support\\Traits\\ForwardsCalls.php(36): Illuminate\\Database\\Eloquent\\Model::throwBadMethodCallException('middleware')
#1 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\database\\Eloquent\\Model.php(1662): Illuminate\\Database\\Eloquent\\Model->forwardCallTo(Object(Illuminate\\Database\\Eloquent\\Builder), 'middleware', Array)
#2 C:\\xampp\\htdocs\\lumenapijwt\\app\\User.php(66): Illuminate\\Database\\Eloquent\\Model->__call('middleware', Array)
#3 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\auth\\EloquentUserProvider.php(183): App\\User->__construct()
#4 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\auth\\EloquentUserProvider.php(170): Illuminate\\Auth\\EloquentUserProvider->createModel()
#5 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\auth\\EloquentUserProvider.php(117): Illuminate\\Auth\\EloquentUserProvider->newModelQuery()
#6 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\tymon\\jwt-auth\\src\\JWTGuard.php(124): Illuminate\\Auth\\EloquentUserProvider->retrieveByCredentials(Array)
#7 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\auth\\AuthManager.php(307): Tymon\\JWTAuth\\JWTGuard->attempt(Array)
#8 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\support\\Facades\\Facade.php(261): Illuminate\\Auth\\AuthManager->__call('attempt', Array)
#9 C:\\xampp\\htdocs\\lumenapijwt\\app\\Http\\Controllers\\AuthController.php(50): Illuminate\\Support\\Facades\\Facade::__callStatic('attempt', Array)
#10 [internal function]: App\\Http\\Controllers\\AuthController->login(Object(Laravel\\Lumen\\Http\\Request))
#11 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\container\\BoundMethod.php(32): call_user_func_array(Array, Array)
#12 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\container\\Util.php(36): Illuminate\\Container\\BoundMethod::Illuminate\\Container\\{closure}()
#13 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\container\\BoundMethod.php(90): Illuminate\\Container\\Util::unwrapIfClosure(Object(Closure))
#14 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\container\\BoundMethod.php(34): Illuminate\\Container\\BoundMethod::callBoundMethod(Object(Laravel\\Lumen\\Application), Array, Object(Closure))
#15 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\container\\Container.php(592): Illuminate\\Container\\BoundMethod::call(Object(Laravel\\Lumen\\Application), Array, Array, NULL)
#16 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(376): Illuminate\\Container\\Container->call(Array, Array)
#17 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(342): Laravel\\Lumen\\Application->callControllerCallable(Array, Array)
#18 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(316): Laravel\\Lumen\\Application->callLumenController(Object(App\\Http\\Controllers\\AuthController), 'login', Array)
#19 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(278): Laravel\\Lumen\\Application->callControllerAction(Array)
#20 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(263): Laravel\\Lumen\\Application->callActionOnArrayBasedRoute(Array)
#21 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(165): Laravel\\Lumen\\Application->handleFoundRoute(Array)
#22 [internal function]: Laravel\\Lumen\\Application->Laravel\\Lumen\\Concerns\\{closure}(Object(Laravel\\Lumen\\Http\\Request))
#23 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Routing\\Pipeline.php(48): call_user_func(Object(Closure), Object(Laravel\\Lumen\\Http\\Request))
#24 C:\\xampp\\htdocs\\lumenapijwt\\app\\Http\\Middleware\\ExampleMiddleware.php(18): Laravel\\Lumen\\Routing\\Pipeline->Laravel\\Lumen\\Routing\\{closure}(Object(Laravel\\Lumen\\Http\\Request))
#25 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\pipeline\\Pipeline.php(167): App\\Http\\Middleware\\ExampleMiddleware->handle(Object(Laravel\\Lumen\\Http\\Request), Object(Closure))
#26 [internal function]: Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Object(Laravel\\Lumen\\Http\\Request))
#27 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Routing\\Pipeline.php(30): call_user_func(Object(Closure), Object(Laravel\\Lumen\\Http\\Request))
#28 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\pipeline\\Pipeline.php(103): Laravel\\Lumen\\Routing\\Pipeline->Laravel\\Lumen\\Routing\\{closure}(Object(Laravel\\Lumen\\Http\\Request))
#29 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(413): Illuminate\\Pipeline\\Pipeline->then(Object(Closure))
#30 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(171): Laravel\\Lumen\\Application->sendThroughPipeline(Array, Object(Closure))
#31 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(108): Laravel\\Lumen\\Application->dispatch(NULL)
#32 C:\\xampp\\htdocs\\lumenapijwt\\public\\index.php(28): Laravel\\Lumen\\Application->run()
#33 {main}
"} 
[2020-03-25 04:35:06] local.ERROR: BadMethodCallException: Call to undefined method App\User::middleware() in C:\xampp\htdocs\lumenapijwt\vendor\illuminate\support\Traits\ForwardsCalls.php:50
Stack trace:
#0 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\support\Traits\ForwardsCalls.php(36): Illuminate\Database\Eloquent\Model::throwBadMethodCallException('middleware')
#1 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\database\Eloquent\Model.php(1662): Illuminate\Database\Eloquent\Model->forwardCallTo(Object(Illuminate\Database\Eloquent\Builder), 'middleware', Array)
#2 C:\xampp\htdocs\lumenapijwt\app\User.php(38): Illuminate\Database\Eloquent\Model->__call('middleware', Array)
#3 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\auth\EloquentUserProvider.php(183): App\User->__construct()
#4 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\auth\EloquentUserProvider.php(170): Illuminate\Auth\EloquentUserProvider->createModel()
#5 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\auth\EloquentUserProvider.php(117): Illuminate\Auth\EloquentUserProvider->newModelQuery()
#6 C:\xampp\htdocs\lumenapijwt\vendor\tymon\jwt-auth\src\JWTGuard.php(124): Illuminate\Auth\EloquentUserProvider->retrieveByCredentials(Array)
#7 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\auth\AuthManager.php(307): Tymon\JWTAuth\JWTGuard->attempt(Array)
#8 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\support\Facades\Facade.php(261): Illuminate\Auth\AuthManager->__call('attempt', Array)
#9 C:\xampp\htdocs\lumenapijwt\app\Http\Controllers\AuthController.php(50): Illuminate\Support\Facades\Facade::__callStatic('attempt', Array)
#10 [internal function]: App\Http\Controllers\AuthController->login(Object(Laravel\Lumen\Http\Request))
#11 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\container\BoundMethod.php(32): call_user_func_array(Array, Array)
#12 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\container\Util.php(36): Illuminate\Container\BoundMethod::Illuminate\Container\{closure}()
#13 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\container\BoundMethod.php(90): Illuminate\Container\Util::unwrapIfClosure(Object(Closure))
#14 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\container\BoundMethod.php(34): Illuminate\Container\BoundMethod::callBoundMethod(Object(Laravel\Lumen\Application), Array, Object(Closure))
#15 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\container\Container.php(592): Illuminate\Container\BoundMethod::call(Object(Laravel\Lumen\Application), Array, Array, NULL)
#16 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(376): Illuminate\Container\Container->call(Array, Array)
#17 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(342): Laravel\Lumen\Application->callControllerCallable(Array, Array)
#18 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(316): Laravel\Lumen\Application->callLumenController(Object(App\Http\Controllers\AuthController), 'login', Array)
#19 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(278): Laravel\Lumen\Application->callControllerAction(Array)
#20 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(263): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#21 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(165): Laravel\Lumen\Application->handleFoundRoute(Array)
#22 [internal function]: Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}(Object(Laravel\Lumen\Http\Request))
#23 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Routing\Pipeline.php(48): call_user_func(Object(Closure), Object(Laravel\Lumen\Http\Request))
#24 C:\xampp\htdocs\lumenapijwt\app\Http\Middleware\ExampleMiddleware.php(18): Laravel\Lumen\Routing\Pipeline->Laravel\Lumen\Routing\{closure}(Object(Laravel\Lumen\Http\Request))
#25 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\pipeline\Pipeline.php(167): App\Http\Middleware\ExampleMiddleware->handle(Object(Laravel\Lumen\Http\Request), Object(Closure))
#26 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Laravel\Lumen\Http\Request))
#27 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Routing\Pipeline.php(30): call_user_func(Object(Closure), Object(Laravel\Lumen\Http\Request))
#28 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\pipeline\Pipeline.php(103): Laravel\Lumen\Routing\Pipeline->Laravel\Lumen\Routing\{closure}(Object(Laravel\Lumen\Http\Request))
#29 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(413): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#30 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(171): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#31 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(108): Laravel\Lumen\Application->dispatch(NULL)
#32 C:\xampp\htdocs\lumenapijwt\public\index.php(28): Laravel\Lumen\Application->run()
#33 {main} {"exception":"[object] (BadMethodCallException(code: 0): Call to undefined method App\\User::middleware() at C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\support\\Traits\\ForwardsCalls.php:50)
[stacktrace]
#0 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\support\\Traits\\ForwardsCalls.php(36): Illuminate\\Database\\Eloquent\\Model::throwBadMethodCallException('middleware')
#1 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\database\\Eloquent\\Model.php(1662): Illuminate\\Database\\Eloquent\\Model->forwardCallTo(Object(Illuminate\\Database\\Eloquent\\Builder), 'middleware', Array)
#2 C:\\xampp\\htdocs\\lumenapijwt\\app\\User.php(38): Illuminate\\Database\\Eloquent\\Model->__call('middleware', Array)
#3 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\auth\\EloquentUserProvider.php(183): App\\User->__construct()
#4 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\auth\\EloquentUserProvider.php(170): Illuminate\\Auth\\EloquentUserProvider->createModel()
#5 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\auth\\EloquentUserProvider.php(117): Illuminate\\Auth\\EloquentUserProvider->newModelQuery()
#6 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\tymon\\jwt-auth\\src\\JWTGuard.php(124): Illuminate\\Auth\\EloquentUserProvider->retrieveByCredentials(Array)
#7 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\auth\\AuthManager.php(307): Tymon\\JWTAuth\\JWTGuard->attempt(Array)
#8 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\support\\Facades\\Facade.php(261): Illuminate\\Auth\\AuthManager->__call('attempt', Array)
#9 C:\\xampp\\htdocs\\lumenapijwt\\app\\Http\\Controllers\\AuthController.php(50): Illuminate\\Support\\Facades\\Facade::__callStatic('attempt', Array)
#10 [internal function]: App\\Http\\Controllers\\AuthController->login(Object(Laravel\\Lumen\\Http\\Request))
#11 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\container\\BoundMethod.php(32): call_user_func_array(Array, Array)
#12 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\container\\Util.php(36): Illuminate\\Container\\BoundMethod::Illuminate\\Container\\{closure}()
#13 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\container\\BoundMethod.php(90): Illuminate\\Container\\Util::unwrapIfClosure(Object(Closure))
#14 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\container\\BoundMethod.php(34): Illuminate\\Container\\BoundMethod::callBoundMethod(Object(Laravel\\Lumen\\Application), Array, Object(Closure))
#15 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\container\\Container.php(592): Illuminate\\Container\\BoundMethod::call(Object(Laravel\\Lumen\\Application), Array, Array, NULL)
#16 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(376): Illuminate\\Container\\Container->call(Array, Array)
#17 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(342): Laravel\\Lumen\\Application->callControllerCallable(Array, Array)
#18 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(316): Laravel\\Lumen\\Application->callLumenController(Object(App\\Http\\Controllers\\AuthController), 'login', Array)
#19 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(278): Laravel\\Lumen\\Application->callControllerAction(Array)
#20 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(263): Laravel\\Lumen\\Application->callActionOnArrayBasedRoute(Array)
#21 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(165): Laravel\\Lumen\\Application->handleFoundRoute(Array)
#22 [internal function]: Laravel\\Lumen\\Application->Laravel\\Lumen\\Concerns\\{closure}(Object(Laravel\\Lumen\\Http\\Request))
#23 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Routing\\Pipeline.php(48): call_user_func(Object(Closure), Object(Laravel\\Lumen\\Http\\Request))
#24 C:\\xampp\\htdocs\\lumenapijwt\\app\\Http\\Middleware\\ExampleMiddleware.php(18): Laravel\\Lumen\\Routing\\Pipeline->Laravel\\Lumen\\Routing\\{closure}(Object(Laravel\\Lumen\\Http\\Request))
#25 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\pipeline\\Pipeline.php(167): App\\Http\\Middleware\\ExampleMiddleware->handle(Object(Laravel\\Lumen\\Http\\Request), Object(Closure))
#26 [internal function]: Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Object(Laravel\\Lumen\\Http\\Request))
#27 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Routing\\Pipeline.php(30): call_user_func(Object(Closure), Object(Laravel\\Lumen\\Http\\Request))
#28 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\pipeline\\Pipeline.php(103): Laravel\\Lumen\\Routing\\Pipeline->Laravel\\Lumen\\Routing\\{closure}(Object(Laravel\\Lumen\\Http\\Request))
#29 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(413): Illuminate\\Pipeline\\Pipeline->then(Object(Closure))
#30 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(171): Laravel\\Lumen\\Application->sendThroughPipeline(Array, Object(Closure))
#31 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(108): Laravel\\Lumen\\Application->dispatch(NULL)
#32 C:\\xampp\\htdocs\\lumenapijwt\\public\\index.php(28): Laravel\\Lumen\\Application->run()
#33 {main}
"} 
[2020-03-25 04:44:27] local.ERROR: ReflectionException: Class App\Http\Controllers\DevSiswaController does not exist in C:\xampp\htdocs\lumenapijwt\vendor\illuminate\container\Container.php:807
Stack trace:
#0 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\container\Container.php(807): ReflectionClass->__construct('App\\Http\\Contro...')
#1 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\container\Container.php(685): Illuminate\Container\Container->build('App\\Http\\Contro...')
#2 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\container\Container.php(633): Illuminate\Container\Container->resolve('App\\Http\\Contro...', Array)
#3 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Application.php(267): Illuminate\Container\Container->make('App\\Http\\Contro...', Array)
#4 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(311): Laravel\Lumen\Application->make('App\\Http\\Contro...')
#5 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(278): Laravel\Lumen\Application->callControllerAction(Array)
#6 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(263): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#7 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(165): Laravel\Lumen\Application->handleFoundRoute(Array)
#8 [internal function]: Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}(Object(Laravel\Lumen\Http\Request))
#9 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Routing\Pipeline.php(48): call_user_func(Object(Closure), Object(Laravel\Lumen\Http\Request))
#10 C:\xampp\htdocs\lumenapijwt\app\Http\Middleware\ExampleMiddleware.php(18): Laravel\Lumen\Routing\Pipeline->Laravel\Lumen\Routing\{closure}(Object(Laravel\Lumen\Http\Request))
#11 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\pipeline\Pipeline.php(167): App\Http\Middleware\ExampleMiddleware->handle(Object(Laravel\Lumen\Http\Request), Object(Closure))
#12 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Laravel\Lumen\Http\Request))
#13 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Routing\Pipeline.php(30): call_user_func(Object(Closure), Object(Laravel\Lumen\Http\Request))
#14 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\pipeline\Pipeline.php(103): Laravel\Lumen\Routing\Pipeline->Laravel\Lumen\Routing\{closure}(Object(Laravel\Lumen\Http\Request))
#15 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(413): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#16 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(171): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#17 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(108): Laravel\Lumen\Application->dispatch(NULL)
#18 C:\xampp\htdocs\lumenapijwt\public\index.php(28): Laravel\Lumen\Application->run()
#19 {main}

Next Illuminate\Contracts\Container\BindingResolutionException: Target class [App\Http\Controllers\DevSiswaController] does not exist. in C:\xampp\htdocs\lumenapijwt\vendor\illuminate\container\Container.php:809
Stack trace:
#0 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\container\Container.php(685): Illuminate\Container\Container->build('App\\Http\\Contro...')
#1 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\container\Container.php(633): Illuminate\Container\Container->resolve('App\\Http\\Contro...', Array)
#2 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Application.php(267): Illuminate\Container\Container->make('App\\Http\\Contro...', Array)
#3 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(311): Laravel\Lumen\Application->make('App\\Http\\Contro...')
#4 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(278): Laravel\Lumen\Application->callControllerAction(Array)
#5 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(263): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#6 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(165): Laravel\Lumen\Application->handleFoundRoute(Array)
#7 [internal function]: Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}(Object(Laravel\Lumen\Http\Request))
#8 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Routing\Pipeline.php(48): call_user_func(Object(Closure), Object(Laravel\Lumen\Http\Request))
#9 C:\xampp\htdocs\lumenapijwt\app\Http\Middleware\ExampleMiddleware.php(18): Laravel\Lumen\Routing\Pipeline->Laravel\Lumen\Routing\{closure}(Object(Laravel\Lumen\Http\Request))
#10 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\pipeline\Pipeline.php(167): App\Http\Middleware\ExampleMiddleware->handle(Object(Laravel\Lumen\Http\Request), Object(Closure))
#11 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Laravel\Lumen\Http\Request))
#12 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Routing\Pipeline.php(30): call_user_func(Object(Closure), Object(Laravel\Lumen\Http\Request))
#13 C:\xampp\htdocs\lumenapijwt\vendor\illuminate\pipeline\Pipeline.php(103): Laravel\Lumen\Routing\Pipeline->Laravel\Lumen\Routing\{closure}(Object(Laravel\Lumen\Http\Request))
#14 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(413): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#15 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(171): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#16 C:\xampp\htdocs\lumenapijwt\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(108): Laravel\Lumen\Application->dispatch(NULL)
#17 C:\xampp\htdocs\lumenapijwt\public\index.php(28): Laravel\Lumen\Application->run()
#18 {main} {"exception":"[object] (Illuminate\\Contracts\\Container\\BindingResolutionException(code: 0): Target class [App\\Http\\Controllers\\DevSiswaController] does not exist. at C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\container\\Container.php:809)
[stacktrace]
#0 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\container\\Container.php(685): Illuminate\\Container\\Container->build('App\\\\Http\\\\Contro...')
#1 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\container\\Container.php(633): Illuminate\\Container\\Container->resolve('App\\\\Http\\\\Contro...', Array)
#2 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Application.php(267): Illuminate\\Container\\Container->make('App\\\\Http\\\\Contro...', Array)
#3 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(311): Laravel\\Lumen\\Application->make('App\\\\Http\\\\Contro...')
#4 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(278): Laravel\\Lumen\\Application->callControllerAction(Array)
#5 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(263): Laravel\\Lumen\\Application->callActionOnArrayBasedRoute(Array)
#6 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(165): Laravel\\Lumen\\Application->handleFoundRoute(Array)
#7 [internal function]: Laravel\\Lumen\\Application->Laravel\\Lumen\\Concerns\\{closure}(Object(Laravel\\Lumen\\Http\\Request))
#8 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Routing\\Pipeline.php(48): call_user_func(Object(Closure), Object(Laravel\\Lumen\\Http\\Request))
#9 C:\\xampp\\htdocs\\lumenapijwt\\app\\Http\\Middleware\\ExampleMiddleware.php(18): Laravel\\Lumen\\Routing\\Pipeline->Laravel\\Lumen\\Routing\\{closure}(Object(Laravel\\Lumen\\Http\\Request))
#10 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\pipeline\\Pipeline.php(167): App\\Http\\Middleware\\ExampleMiddleware->handle(Object(Laravel\\Lumen\\Http\\Request), Object(Closure))
#11 [internal function]: Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Object(Laravel\\Lumen\\Http\\Request))
#12 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Routing\\Pipeline.php(30): call_user_func(Object(Closure), Object(Laravel\\Lumen\\Http\\Request))
#13 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\pipeline\\Pipeline.php(103): Laravel\\Lumen\\Routing\\Pipeline->Laravel\\Lumen\\Routing\\{closure}(Object(Laravel\\Lumen\\Http\\Request))
#14 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(413): Illuminate\\Pipeline\\Pipeline->then(Object(Closure))
#15 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(171): Laravel\\Lumen\\Application->sendThroughPipeline(Array, Object(Closure))
#16 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(108): Laravel\\Lumen\\Application->dispatch(NULL)
#17 C:\\xampp\\htdocs\\lumenapijwt\\public\\index.php(28): Laravel\\Lumen\\Application->run()
#18 {main}

[previous exception] [object] (ReflectionException(code: -1): Class App\\Http\\Controllers\\DevSiswaController does not exist at C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\container\\Container.php:807)
[stacktrace]
#0 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\container\\Container.php(807): ReflectionClass->__construct('App\\\\Http\\\\Contro...')
#1 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\container\\Container.php(685): Illuminate\\Container\\Container->build('App\\\\Http\\\\Contro...')
#2 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\container\\Container.php(633): Illuminate\\Container\\Container->resolve('App\\\\Http\\\\Contro...', Array)
#3 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Application.php(267): Illuminate\\Container\\Container->make('App\\\\Http\\\\Contro...', Array)
#4 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(311): Laravel\\Lumen\\Application->make('App\\\\Http\\\\Contro...')
#5 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(278): Laravel\\Lumen\\Application->callControllerAction(Array)
#6 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(263): Laravel\\Lumen\\Application->callActionOnArrayBasedRoute(Array)
#7 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(165): Laravel\\Lumen\\Application->handleFoundRoute(Array)
#8 [internal function]: Laravel\\Lumen\\Application->Laravel\\Lumen\\Concerns\\{closure}(Object(Laravel\\Lumen\\Http\\Request))
#9 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Routing\\Pipeline.php(48): call_user_func(Object(Closure), Object(Laravel\\Lumen\\Http\\Request))
#10 C:\\xampp\\htdocs\\lumenapijwt\\app\\Http\\Middleware\\ExampleMiddleware.php(18): Laravel\\Lumen\\Routing\\Pipeline->Laravel\\Lumen\\Routing\\{closure}(Object(Laravel\\Lumen\\Http\\Request))
#11 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\pipeline\\Pipeline.php(167): App\\Http\\Middleware\\ExampleMiddleware->handle(Object(Laravel\\Lumen\\Http\\Request), Object(Closure))
#12 [internal function]: Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}(Object(Laravel\\Lumen\\Http\\Request))
#13 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Routing\\Pipeline.php(30): call_user_func(Object(Closure), Object(Laravel\\Lumen\\Http\\Request))
#14 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\illuminate\\pipeline\\Pipeline.php(103): Laravel\\Lumen\\Routing\\Pipeline->Laravel\\Lumen\\Routing\\{closure}(Object(Laravel\\Lumen\\Http\\Request))
#15 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(413): Illuminate\\Pipeline\\Pipeline->then(Object(Closure))
#16 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(171): Laravel\\Lumen\\Application->sendThroughPipeline(Array, Object(Closure))
#17 C:\\xampp\\htdocs\\lumenapijwt\\vendor\\laravel\\lumen-framework\\src\\Concerns\\RoutesRequests.php(108): Laravel\\Lumen\\Application->dispatch(NULL)
#18 C:\\xampp\\htdocs\\lumenapijwt\\public\\index.php(28): Laravel\\Lumen\\Application->run()
#19 {main}
"} 
